<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.361">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<meta name="author" content="Gilberto Camara, Lorenzo de Simone, Ronald Jansen">
<title>UN Handbook on Remote Sensing for Agricultural Statistics - 18&nbsp; Crop classification and land use mapping in Chile</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>

<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./ct_digital_earth_africa.html" rel="next">
<link href="./ct_zimbabwe.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light"><script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
</head>
<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top"><nav class="quarto-secondary-nav"><div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./crop_type_mapping.html">Use Cases in Crop Type Mapping</a></li><li class="breadcrumb-item"><a href="./ct_chile.html"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Crop classification and land use mapping in Chile</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav></header><!-- content --><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto"><div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">UN Handbook on Remote Sensing for Agricultural Statistics</a> 
        <div class="sidebar-tools-main">
    <a href="https://github.com/FAO-EOSTAT/UN-Handbook" title="Source Code" class="quarto-navigation-tool px-1" aria-label="Source Code"><i class="bi bi-github"></i></a>
  <a href="" class="quarto-reader-toggle quarto-navigation-tool px-1" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Welcome</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./howto.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">How to use this handbook</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Introduction</span></span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./theory.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Foundations</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_remote_sensing.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Remote sensing images: optical, SAR</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_data_sources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Earth observation big data sources</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_data_cubes.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Earth observation data cubes</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_lucc.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Land cover and crop classification schemas</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_quality_control.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Quality control of training sets for agricultural statistics</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_machine_learning.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Machine learning algorithms for image time series</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_uncertainty.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Spatial map uncertainty estimation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_validation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Map validation and use of maps for area estimation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_design_frames.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Remote Sensing in the Design of Sampling Frames</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./th_parcel_extraction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Automatic Extraction of Parcels</span></span></a>
  </div>
</li>
      </ul>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./crop_type_mapping.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Use Cases in Crop Type Mapping</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ct_poland.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Crop monitoring with Sentinel-1 and Sentinel-2 images in Poland</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ct_mexico.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Crop classification in Mexico</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ct_senegal.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Multi-seasonal crop mapping in Senegal</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ct_zimbabwe.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Crop classification in Zimbabwe</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ct_chile.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Crop classification and land use mapping in Chile</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ct_digital_earth_africa.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Agricultural Mapping with Digital Earth Africa</span></span></a>
  </div>
</li>
      </ul>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./crop_yield_estimation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Crop yield estimation</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cy_finland.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Early-season crop yield mapping in Finland</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cy_indonesia.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Mapping crop phenology in Indonesia</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cy_poland.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Yield forecasting in Poland</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cy_colombia.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Rice Phenology in Colombia</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cy_china.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Crop type classification and crop yield estimation in China</span></span></a>
  </div>
</li>
      </ul>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./additional.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Additional Topics</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ad_geoglam.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">25</span>&nbsp; <span class="chapter-title">Extraction of crop statistics from crop type and crop yield maps</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ad_world_cereal.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">26</span>&nbsp; <span class="chapter-title">WorldCereal - A Global Effort for Crop Mapping</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ad_uav_applications.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">27</span>&nbsp; <span class="chapter-title">UAV use in Agricultural Statistics</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ad_disaster_response.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">28</span>&nbsp; <span class="chapter-title">Remote Sensing for Agricultural Disaster Response</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ad_china_floods.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">29</span>&nbsp; <span class="chapter-title">An Automated Flood Detection Algorithm for Post-Disaster Assessment</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ad_governance.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">30</span>&nbsp; <span class="chapter-title">Data Governance for Agricultural Statistics</span></span></a>
  </div>
</li>
      </ul>
</li>
    </ul>
</div>
</nav><div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active"><h2 id="toc-title">Table of contents</h2>
   
  <ul>
<li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">18.1</span> Introduction</a></li>
  <li>
<a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods"><span class="header-section-number">18.2</span> Methods</a>
  <ul class="collapse">
<li><a href="#study-area" id="toc-study-area" class="nav-link" data-scroll-target="#study-area"><span class="header-section-number">18.2.1</span> Study area</a></li>
  <li><a href="#classification-scheme" id="toc-classification-scheme" class="nav-link" data-scroll-target="#classification-scheme"><span class="header-section-number">18.2.2</span> Classification scheme</a></li>
  <li><a href="#sec-training-data" id="toc-sec-training-data" class="nav-link" data-scroll-target="#sec-training-data"><span class="header-section-number">18.2.3</span> Training data</a></li>
  <li><a href="#generation-of-land-use-and-land-cover-maps-using-time-series-of-satellite-images" id="toc-generation-of-land-use-and-land-cover-maps-using-time-series-of-satellite-images" class="nav-link" data-scroll-target="#generation-of-land-use-and-land-cover-maps-using-time-series-of-satellite-images"><span class="header-section-number">18.2.4</span> Generation of land use and land cover maps using time series of satellite images</a></li>
  </ul>
</li>
  <li><a href="#configuration-to-run-the-example-in-this-chapter" id="toc-configuration-to-run-the-example-in-this-chapter" class="nav-link" data-scroll-target="#configuration-to-run-the-example-in-this-chapter">Configuration to run the example in this chapter</a></li>
  <li><a href="#study-area-data" id="toc-study-area-data" class="nav-link" data-scroll-target="#study-area-data"><span class="header-section-number">18.3</span> Study area data</a></li>
  <li>
<a href="#earth-observation-data-cubes" id="toc-earth-observation-data-cubes" class="nav-link" data-scroll-target="#earth-observation-data-cubes"><span class="header-section-number">18.4</span> Earth observation data cubes</a>
  <ul class="collapse">
<li><a href="#accessing-sentinel-22a-images" id="toc-accessing-sentinel-22a-images" class="nav-link" data-scroll-target="#accessing-sentinel-22a-images">Accessing SENTINEL-2/2A images</a></li>
  <li><a href="#copernicus-dem-30-meter-images" id="toc-copernicus-dem-30-meter-images" class="nav-link" data-scroll-target="#copernicus-dem-30-meter-images">Copernicus DEM 30 meter images</a></li>
  <li><a href="#computing-vegetation-indexes" id="toc-computing-vegetation-indexes" class="nav-link" data-scroll-target="#computing-vegetation-indexes">Computing vegetation indexes</a></li>
  <li><a href="#combining-multitemporal-data-cubes-with-dems" id="toc-combining-multitemporal-data-cubes-with-dems" class="nav-link" data-scroll-target="#combining-multitemporal-data-cubes-with-dems">Combining multitemporal data cubes with DEMs</a></li>
  </ul>
</li>
  <li>
<a href="#satellite-image-time-series" id="toc-satellite-image-time-series" class="nav-link" data-scroll-target="#satellite-image-time-series"><span class="header-section-number">18.5</span> Satellite image time series</a>
  <ul class="collapse">
<li><a href="#getting-time-series-data-from-data-cubes" id="toc-getting-time-series-data-from-data-cubes" class="nav-link" data-scroll-target="#getting-time-series-data-from-data-cubes">Getting time series data from data cubes</a></li>
  </ul>
</li>
  <li><a href="#training-a-machine-learning-model" id="toc-training-a-machine-learning-model" class="nav-link" data-scroll-target="#training-a-machine-learning-model"><span class="header-section-number">18.6</span> Training a machine learning model</a></li>
  <li><a href="#sec-data-cube-classification" id="toc-sec-data-cube-classification" class="nav-link" data-scroll-target="#sec-data-cube-classification"><span class="header-section-number">18.7</span> Data cube classification</a></li>
  <li><a href="#spatial-smoothing" id="toc-spatial-smoothing" class="nav-link" data-scroll-target="#spatial-smoothing"><span class="header-section-number">18.8</span> Spatial smoothing</a></li>
  <li><a href="#sec-labeling-a-probability-data-cube" id="toc-sec-labeling-a-probability-data-cube" class="nav-link" data-scroll-target="#sec-labeling-a-probability-data-cube"><span class="header-section-number">18.9</span> Labeling a probability data cube</a></li>
  <li>
<a href="#validation-and-accuracy-assessment-of-the-map" id="toc-validation-and-accuracy-assessment-of-the-map" class="nav-link" data-scroll-target="#validation-and-accuracy-assessment-of-the-map"><span class="header-section-number">18.10</span> Validation and accuracy assessment of the map</a>
  <ul class="collapse">
<li><a href="#sampling-design" id="toc-sampling-design" class="nav-link" data-scroll-target="#sampling-design"><span class="header-section-number">18.10.1</span> Sampling design</a></li>
  <li><a href="#sec-stratified-random-sampling" id="toc-sec-stratified-random-sampling" class="nav-link" data-scroll-target="#sec-stratified-random-sampling"><span class="header-section-number">18.10.2</span> Stratified random sampling</a></li>
  <li><a href="#accuracy-assessment-of-classified-sampling" id="toc-accuracy-assessment-of-classified-sampling" class="nav-link" data-scroll-target="#accuracy-assessment-of-classified-sampling"><span class="header-section-number">18.10.3</span> Accuracy assessment of classified sampling</a></li>
  </ul>
</li>
  <li>
<a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">18.11</span> Discussion</a>
  <ul class="collapse">
<li><a href="#institutional-relevance-and-purpose-of-the-map" id="toc-institutional-relevance-and-purpose-of-the-map" class="nav-link" data-scroll-target="#institutional-relevance-and-purpose-of-the-map">Institutional relevance and purpose of the map</a></li>
  <li><a href="#classification-and-sampling-scheme-in-lulc-mapping" id="toc-classification-and-sampling-scheme-in-lulc-mapping" class="nav-link" data-scroll-target="#classification-and-sampling-scheme-in-lulc-mapping">Classification and sampling scheme in LULC mapping</a></li>
  <li><a href="#construction-of-macrozones-as-a-basis-for-territorial-modeling" id="toc-construction-of-macrozones-as-a-basis-for-territorial-modeling" class="nav-link" data-scroll-target="#construction-of-macrozones-as-a-basis-for-territorial-modeling">Construction of macrozones as a basis for territorial modeling</a></li>
  <li><a href="#inclusion-of-the-dem-as-a-feature-for-modeling" id="toc-inclusion-of-the-dem-as-a-feature-for-modeling" class="nav-link" data-scroll-target="#inclusion-of-the-dem-as-a-feature-for-modeling">Inclusion of the DEM as a feature for modeling</a></li>
  <li><a href="#analysis-of-the-quality-of-training-data" id="toc-analysis-of-the-quality-of-training-data" class="nav-link" data-scroll-target="#analysis-of-the-quality-of-training-data">Analysis of the quality of training data</a></li>
  <li><a href="#model-evaluation" id="toc-model-evaluation" class="nav-link" data-scroll-target="#model-evaluation">Model evaluation</a></li>
  <li><a href="#advanced-applications-of-remote-sensing-and-deep-learning-in-official-statistics" id="toc-advanced-applications-of-remote-sensing-and-deep-learning-in-official-statistics" class="nav-link" data-scroll-target="#advanced-applications-of-remote-sensing-and-deep-learning-in-official-statistics">Advanced applications of remote sensing and deep learning in official statistics</a></li>
  <li><a href="#limitations-and-opportunities-for-improvement" id="toc-limitations-and-opportunities-for-improvement" class="nav-link" data-scroll-target="#limitations-and-opportunities-for-improvement">Limitations and opportunities for improvement</a></li>
  </ul>
</li>
  <li><a href="#conclusions" id="toc-conclusions" class="nav-link" data-scroll-target="#conclusions"><span class="header-section-number">18.12</span> Conclusions</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul><div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/FAO-EOSTAT/UN-Handbook/edit/quarto/ct_chile.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/FAO-EOSTAT/UN-Handbook/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block default"><div class="quarto-title">
<h1 class="title">
<span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Crop classification and land use mapping in Chile</span>
</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Authors</div>
    <div class="quarto-title-meta-contents">
             <p>Estefanía Pizarro Arias </p>
             <p>Alexis Gamboa Maldonado </p>
             <p>Jonathan Paredes Baez </p>
          </div>
  </div>
    
  
    
  </div>
  

</header><section id="introduction" class="level2" data-number="18.1"><h2 data-number="18.1" class="anchored" data-anchor-id="introduction">
<span class="header-section-number">18.1</span> Introduction</h2>
<p>The National Institute of Statistics (INE) of Chile is the technical agency responsible for producing, collecting, processing, and disseminating official statistics. Its primary mission is to provide timely, objective, and high-quality statistical information to support public and private decision-making, public policy design, territorial planning, and socioeconomic analysis. In fulfilling this mission, the INE adheres to principles of transparency, impartiality, methodological rigor, and confidentiality, aligned with international standards set by the United Nations and other specialized organizations. It also continuously improves its processes through technological modernization, methodology validation, and ongoing staff training initiatives (<a href="https://www.ine.gob.cl/institucional/plan-estrategico-2022-2026/mision-vision-institucional-y-vision-superior">INE</a>).</p>
<p>Despite these ongoing efforts, a significant gap remains: the absence of a standardized Land Use and Land Cover (LULC) map specifically designed to support the production of official agricultural statistics. In Chile, various LULC maps have been developed; however, none have been explicitly tailored for this purpose <span class="citation" data-cites="INE2025"><a href="#ref-INE2025" role="doc-biblioref">[1]</a></span>. This situation presents multiple challenges, including the lack of standardized information, limited data availability, and difficulties integrating existing data into official statistical production. To overcome these limitations, the INE has initiated the development of a national LULC map, aiming to optimize the primary database used for the construction and updating of the sampling frame for intercensal agricultural surveys. This initiative aims to significantly enhance the precision, relevance, and timeliness of agricultural statistics, enabling better informed decision and more efficient planning.</p>
<p>The benefits of developing a LULC map are diverse. In terms of the Agricultural Master Sampling Frame, it will ensure the timely availability of updated inputs, promote the construction and maintenance of the sampling frame under standardized criteria, support the updating of stratification processes, and guarantee exhaustive coverage to minimize and properly correct sampling biases <span class="citation" data-cites="Gallego2017"><a href="#ref-Gallego2017" role="doc-biblioref">[2]</a></span>. From the perspective of agricultural statistics production, the LULC map will serve as a complementary source of information essential for estimating areas of productive land cover, monitoring temporal changes in land use, providing key indicators for remote or inaccessible areas, and improving the efficiency of resource use by reducing the costs associated with traditional data collection methods <span class="citation" data-cites="Delince2017"><a href="#ref-Delince2017" role="doc-biblioref">[3]</a></span>. The relevance of this effort becomes even greater when considering the inherent challenges of agricultural statistics. The production landscape is highly dynamic, influenced by local management decisions, meteorological variability, and natural disasters, which generate short- and medium-term changes <span class="citation" data-cites="Davidson2017"><a href="#ref-Davidson2017" role="doc-biblioref">[4]</a></span>. Maintaining continuous and reliable information on a national scale under these conditions entails high economic costs. Thus, advancing the adoption of remote sensing technologies, such as satellite imagery or drone-based surveys, is essential to optimize and reinforce field data collection efforts for agricultural statistics.</p>
<p>LULC maps generated with Earth Observation (EO) data can play a key role in supporting sample design through improved stratification. For instance, crop intensity maps can significantly reduce sampling variances or lower field data collection efforts and associated costs <span class="citation" data-cites="Defourny2017"><a href="#ref-Defourny2017" role="doc-biblioref">[5]</a></span>. Moreover, identifying non-agricultural strata that should not be sampled, or that require differentiated treatment, can enhance the efficiency of statistical surveys. However, the efficiency of stratification depends critically on the relevance and quality of the employed land cover map.</p>
<p>Recent advancements in EO technology have markedly improved the spatiotemporal monitoring of agricultural surfaces. These technological advances enable more timely and continuous assessments of changes in agricultural land use, including the expansion of cropland into natural vegetation and the reduction of agricultural areas due to droughts, floods, or wildfires <span class="citation" data-cites="Davidson2017"><a href="#ref-Davidson2017" role="doc-biblioref">[4]</a></span>. Given these advancements, the use of EO-derived LULC maps emerges as a fundamental tool for modernizing and enhancing the production of official agricultural statistics in Chile. It offers a standardized, timely, and efficient approach to understanding agricultural dynamics, improving statistical accuracy, and supporting evidence-based policy and planning.</p>
<p>This study applies and builds upon the methodological improvements and lessons learned from the pilot project carried out in the Maule Region in 2024 by INE Chile. This project established the groundwork for using land use and land cover classification processes based on satellite and auxiliary data as an input to produce agricultural statistics <span class="citation" data-cites="INE2025"><a href="#ref-INE2025" role="doc-biblioref">[1]</a></span>.</p>
</section><section id="methods" class="level2" data-number="18.2"><h2 data-number="18.2" class="anchored" data-anchor-id="methods">
<span class="header-section-number">18.2</span> Methods</h2>
<section id="study-area" class="level3" data-number="18.2.1"><h3 data-number="18.2.1" class="anchored" data-anchor-id="study-area">
<span class="header-section-number">18.2.1</span> Study area</h3>
<p>One of the main challenges of this work is the territorial extent of Chile, the longest country in the world, spanning more than 4,200 kilometers from north to south and located approximately between latitudes 18° and 56° South. This geographic configuration results in high climatic variability <span class="citation" data-cites="Sarricolea2017"><a href="#ref-Sarricolea2017" role="doc-biblioref">[6]</a></span> and a wide diversity of landscapes and land-forms, which in turn condition the presence of various land cover types and land use practices</p>
<p>In our case, we developed a proposal for subdividing continental Chile into macrozones, defined as groupings of provinces (political-administrative units within regions) with similar geographical characteristics (understood as those climatic, vegetational, and agricultural variables that determine the landscape of a given territory). To support this, three sources of geographical characteristics for Chile were considered: (1) the Chilean Agroclimatic Atlas <span class="citation" data-cites="Santibanez2017"><a href="#ref-Santibanez2017" role="doc-biblioref">[7]</a></span>, (2) the Bioclimatic and Vegetational Synopsis of Chile <span class="citation" data-cites="Luebert2017"><a href="#ref-Luebert2017" role="doc-biblioref">[8]</a></span>, and (3) the Rural Atlas of Chile <span class="citation" data-cites="Odepa2022b"><a href="#ref-Odepa2022b" role="doc-biblioref">[9]</a></span>.</p>
<p>These three sources were analyzed spatially by identifying the territorial distribution of each classification, and then grouping, from north to south, those provinces with similar characteristics. Thus, we arrived at a proposal of nine macrozones that group the country’s 56 provinces based on similar geographical characteristics. In this case, we focused on Macrozone 4 (MZ4), which includes the provinces of the Maule and Biobío regions.</p>
<p>MZ4 is located between latitudes 34°41’ and 37°18’ south (<a href="#fig-map">Figure&nbsp;<span>18.1</span></a>) and covers an area of 43,394 km<sup>2</sup> (approximately 5.7% of Chile’s surface area), it is characterized by a temperate Mediterranean climate with marked seasonality in precipitation. According to the updated Köppen-Geiger classification <span class="citation" data-cites="Beck2018"><a href="#ref-Beck2018" role="doc-biblioref">[10]</a></span>, the predominant climate in this area is Csb, corresponding to a dry-summer, temperate Mediterranean climate, with precipitation concentrated in the winter months (June to August) and a water deficit in the summer (December to February). In the highest areas of the foothills and the Andes Mountain Range, the climate changes to a cool-summer oceanic regime (Cfb), characterized by higher annual precipitation and lower temperature amplitudes. Its topography includes the country’s main land-forms (Andes Mountain Range, intermediate depression, coastal mountain range, coastal plains, and river basins), which gives it great geomorphological, climatic, and ecological diversity. Its topography includes mountains reaching near 4.000 m.a.s.l. in the Andes.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-map" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/Map_MZ4_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.1: Map of the Macrozone in study (Source: National Institute of Statistics of Chile with data from Conaf<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>).</figcaption></figure>
</div>
</div>
</div>
<p>According to the 2021 Census of Agriculture (<a href="https://www.ine.gob.cl/docs/default-source/censo-agropecuario/cuadros-estadisticos/2021/numero-de-upa-y-superficie-por-categor%C3%ADa-de-uso-del-suelo.xlsx?sfvrsn=3ea7240d_4">INE</a>), the MZ4 plays a significant role in Chile´s agricultural and forestry production. Based on the censuses of land categories, the primary uses of land in the zone are croplands and forest plantations (50.5%), forest (19.4%), and grassland-shrubland (18.9%). Additionally, this zone represents 27.8% of the national area of croplands and forest plantations (considering the census land area). In this context, land use in the zone is primarily focused on forest plantations (60.9%), cereals (14.5%), fruit orchards (11.4%), and vineyards (4.4%), totaling 91.2% of the area’s zone. This figures underscores the importance of the MZ4 in Chile’s agricultural and forestry production.</p>
</section><section id="classification-scheme" class="level3" data-number="18.2.2"><h3 data-number="18.2.2" class="anchored" data-anchor-id="classification-scheme">
<span class="header-section-number">18.2.2</span> Classification scheme</h3>
<p>During the development of the pilot mapping of LULC in the Maule region, a classification scheme built on predominantly agronomic criteria was used. It grouped the coverages according to their predominant use (temporary crops, permanent crops, pastures, etc.), without incorporating explicit temporal or phenological attributes <span class="citation" data-cites="INE2025"><a href="#ref-INE2025" role="doc-biblioref">[1]</a></span>.</p>
<p>However, this methodological approach presented limitations when applying algorithms to time series of satellite images. Despite being defined as separate classes in the classification scheme, the spectral and phenological similarity, such as in the case of improved pastures and forage crops, often led to confusion during the classification process. These limitations were considered in a new proposal.</p>
<p>A new classification scheme was created maintaining a hierarchical structure despite technical limitations and institutional requirements <span class="citation" data-cites="Ojwang2024"><a href="#ref-Ojwang2024" role="doc-biblioref">[11]</a></span> but incorporating with greater emphasis the phenological dimension. This new multi-level structure allowed a more precise thematic breakdown (for example, winter crops, successive crops, fallow land), and integrated observable variables in vegetation curves such as the NDVI. The new scheme recognizes the dynamic nature of certain classes, replacing the fixed coverage approach with a temporarily explicit one.</p>
<p>A key challenge in producing annual land cover maps is to properly represent classes that experience seasonal changes within the year. The hierarchical-phenological schemes address this problem by including explicit temporal or seasonal classes, which allow them to represent dynamic coverages without forcing them into static categories <span class="citation" data-cites="Chowdhury2023 Myers2024"><a href="#ref-Chowdhury2023" role="doc-biblioref">[12]</a>, <a href="#ref-Myers2024" role="doc-biblioref">[13]</a></span>.</p>
<p>In our case, new seasonal classes were added to some categories. For example, in native forests they explicitly differentiate between deciduous and perennial trees, integrating phenology as a classification criterion. This differentiation makes it possible, for example, to prevent deciduous forests in winter images from being misclassified as bare soil or disturbed areas. This logic has been formally incorporated into systems such as the FAO’s Land Cover Classification System (LCCS), which explicitly distinguishes between deciduous and perennial forests.</p>
<p>The use of time classes in agricultural areas can be particularly advantageous for representing different crop cycles observed in a single agricultural year. A phenological approach distinguishes between winter, summer, succession, or double harvest crops, unlike traditional schemes that classify crops in aggregate form. It is possible to identify double crop fields by analyzing the number of phenological peaks in the NDVI time series, as demonstrated by <span class="citation" data-cites="Guo2022"><a href="#ref-Guo2022" role="doc-biblioref">[14]</a></span>. The capture of land use intensity by these time classes is essential for agricultural planning, production estimation, and water management.</p>
<p>The integration of multi-time data is essential to represent the dynamics of land cover and use. Several studies have shown that the use of satellite images from multiple dates significantly increases the accuracy in the classification of dynamic coverage <span class="citation" data-cites="Oetter2001 Wardlow2008"><a href="#ref-Oetter2001" role="doc-biblioref">[15]</a>, <a href="#ref-Wardlow2008" role="doc-biblioref">[16]</a></span>. In agricultural landscapes, the combination of summer and winter images allows to separate crops that could be spectrally indistinguishable at single dates. In deciduous forests, the contrast between full-leaf and no-foliage images is crucial to correctly identify their type <span class="citation" data-cites="Oetter2001"><a href="#ref-Oetter2001" role="doc-biblioref">[15]</a></span>.</p>
<p>Additionally, the utilization of phenological data enhances the detection of changes or disturbances. Time series allows for the identification of breaks in expected vegetation behavior, while static approaches tend to omit short-term events such as fires or crops. These ruptures, visible as phenological anomalies, are key to detecting real changes in soil cover. Thus, the hierarchical-phenological scheme not only allows to classify coverage but also acts as a tool for territorial monitoring <span class="citation" data-cites="Zhu2014"><a href="#ref-Zhu2014" role="doc-biblioref">[17]</a></span>.</p>
<p>In this context, a hierarchical-phenological classification approach is proposed, whose structure incorporates the seasonal dynamism of those cover and land use classes that present phenological variations during the year, such as agricultural lands, forest plantations, grassland, and snow-covered areas. <a href="#fig-scheme">Figure&nbsp;<span>18.2</span></a> shows the complete structure of the scheme at three levels.</p>
<p>In the case of agricultural land category, four major classes are distinguished at the second level: temporary crops, permanent crops, pastures, and fallow land. In turn, temporary crops are subdivided into winter, spring/summer, and successive crops. Permanent crops include deciduous and persistent fruit. Annual and perennial pastures are also differentiated, and fallow land is incorporated as a specific category. This hierarchical phenological structure allows a progressive classification based on the phenological dynamics observed in agricultural systems <span class="citation" data-cites="Zhou2024"><a href="#ref-Zhou2024" role="doc-biblioref">[18]</a></span>.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-scheme" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/classification_scheme.jpg" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.2: Classification scheme used for the MZ4 (Source: National Institute of Statistics, Chile).</figcaption></figure>
</div>
</div>
</div>
</section><section id="sec-training-data" class="level3" data-number="18.2.3"><h3 data-number="18.2.3" class="anchored" data-anchor-id="sec-training-data">
<span class="header-section-number">18.2.3</span> Training data</h3>
<p>Field surveys have a limited number of samples, and obtaining many necessitates a significant investment in manpower, financial resources, and time. Therefore, a sampling of points generated from polygons was chosen to increase the number of samples, with the aim of improving the stability and generalization capability of the classifier <span class="citation" data-cites="Zhou2019"><a href="#ref-Zhou2019" role="doc-biblioref">[19]</a></span>. The polygons were constructed as training sites for each of the 24 classes of level three defined in the classification scheme.</p>
<p>As a result, numerous point-level samples were obtained (near 62,000 labeled points), unlike the polygon samples (4,140 labeled polygons), each with its corresponding time curve and coverage class, derived from the crossover between these points and the satellite data cube. Sampling was designed according to the area distribution of all polygons, assigning each one a minimum and maximum number of points, determined in proportion to their relative size versus the rest.</p>
<p>According to <a href="#fig-diag">Figure&nbsp;<span>18.3</span></a>, the process begins with the establishment of a hierarchical classification based on the phenology of the different categories of land cover and use. Data from agricultural censuses and cadastres are used to define the classes. Subsequently, each class is validated by temporal analysis of NDVI indices and seasonal RGB compositions, complemented with expert interpretation of very high resolution (VHR) images <span class="citation" data-cites="INE2025"><a href="#ref-INE2025" role="doc-biblioref">[1]</a></span>. Finally, sample polygons labeled with their respective classes are digitized, within which random points are generated in quantity proportional to their surface area. These points inherit the label of the corresponding polygon, making them suitable for use as training data in classification models.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-diag" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/training_data_scheme.jpg" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.3: Diagram of the proposed methodological flow for generating training data (Source: National Institute of Statistics, Chile).</figcaption></figure>
</div>
</div>
</div>
<p>The following steps summarize the applied process:</p>
<ol type="1">
<li><p><strong>Definition of classification scheme:</strong> A hierarchical-phenological scheme composed of 24 classes at level 3 was established, defined from the analysis of the different land cover and uses. This analysis considered especially the dynamic and seasonal behavior of many of these classes, allowing a more precise differentiation based on their phenology<strong>.</strong></p></li>
<li>
<p><strong>Construction of coverage and land use polygons using census and cadastral data</strong></p>
<ul>
<li>
<strong>Agricultural census data:</strong> Agricultural and forestry census records are incorporated as a source of reference information. These data include GPS coordinates of the management point performed and not the explicit location of the crop or forest species, so certain filters and assumptions must be applied to make use of the information <span class="citation" data-cites="INE2025"><a href="#ref-INE2025" role="doc-biblioref">[1]</a></span>.</li>
</ul>
<!-- --><ul>
<li>
<strong>Official cadastral data:</strong> Cadastral layers are used as an auxiliary source for delineating land cover. These layers are stable elements and widely available in geographic information systems. Thus, official land boundary information is leveraged to define potential areas of agricultural use, forest zones, natural vegetation, urban settlements, among others, and to assign initial land cover classes <span class="citation" data-cites="INE2025"><a href="#ref-INE2025" role="doc-biblioref">[1]</a></span>
</li>
</ul>
</li>
<li>
<p><strong>Validation of land cover and land use polygons through remote sensing analysis</strong></p>
<ul>
<li>
<strong>NDVI time series:</strong> The use of NDVI time series enables the characterization of the phenological phases of crops and land covers through phenograms that reflect their seasonal dynamics. This information supports a hierarchical classification, first distinguishing between cultivated and non-cultivated areas, and subsequently among specific crop types, by validating the assigned labels based on the observed temporal consistency <span class="citation" data-cites="Tahir2024"><a href="#ref-Tahir2024" role="doc-biblioref">[20]</a></span>.</li>
</ul>
<!-- --><ul>
<li>
<strong>Multitemporal RGB composites:</strong> Multitemporal RGB composites, based on optical imagery from different seasons of the year, highlight seasonal variations in reflectance, facilitating the identification of crops through changes in color and texture. The combination of winter and summer images has proven effective in improving class discrimination by capturing complementary structural and phenological features <span class="citation" data-cites="Ullah2025"><a href="#ref-Ullah2025" role="doc-biblioref">[21]</a></span>.</li>
</ul>
<!-- --><ul>
<li>
<strong>VHR image interpretation:</strong> Validation using very high-resolution (VHR) satellite imagery allows for the confirmation or correction of land cover labels through expert visual interpretation. This technique considered a standard in remote sensing studies, facilitates the verification of reference data based on visual elements such as crop patterns or spatial vegetation structures <span class="citation" data-cites="Schepaschenko2019"><a href="#ref-Schepaschenko2019" role="doc-biblioref">[22]</a></span>.</li>
</ul>
</li>
<li>
<p><strong>Labeled data generation</strong></p>
<ul>
<li>
<strong>Digitization of representative polygons:</strong> Training data generation is carried out through the manual digitization of polygons over high-resolution imagery, assigning labels corresponding to the Level 3 hierarchical classes defined in the classification scheme. This practice, widely used in remote sensing, enables adequate representation of spatial variability and helps reduce noise associated with individual pixels <span class="citation" data-cites="Elmes2020a"><a href="#ref-Elmes2020a" role="doc-biblioref">[23]</a></span>.</li>
</ul>
<!-- --><ul>
<li>
<p><strong>Consolidation of representative polygons:</strong> The generated polygons are consolidated into a single spatial layer, applying the necessary geometric corrections and eliminating any potential overlaps among them. As we mentioned previously, we obtained 4,140 labeled polygons.</p>
<!-- -->
</li>
<li>
<p><strong>Random sampling of points proportional to polygon area:</strong> Finally, based on the labeled polygons, random sampling points are generated in quantities proportional to the area of each polygon. These points inherit the hierarchical classification of the area in which they are located, enabling the construction of a statistically representative training dataset for each class with low spatial autocorrelation among samples. This dataset is then used by machine learning models, such as Random Forest or neural networks, to classify larger areas of the territory.</p>
<p>Point sampling generation is a process designed to obtain the most representative coverage of the built polygons for each class, considering as the main criterion the area of each of them. To do this, a procedure was implemented that allows you to control different parameters associated with the generation of points from polygons, such as the minimum and maximum number of points per unit, and the minimum distance between points, among others. In the first stage, a frequency analysis of the surfaces of the polygons was carried out, which showed a high variability, with values ranging from 150 to 3,800,000 m². This finding reinforced the methodological decision to establish a proportional allocation of points according to the size of each polygon. After preliminary testing, minimum and maximum ranges of 5 to 25 points per polygon were established. In addition, polygons with surface areas greater than 250,000 m² were excluded from the sampling process due to their disproportionate influence on the overall sample distribution (once the filter is applied, we work with 3,924 labeled polygons). Finally, to control the total number of points generated, a maximum of 5,000 points per class was set, applying a stratified random sampling approach to ensure both representativeness and class balance.</p>
</li>
</ul>
</li>
</ol>
<p>Finally, random sampling of points proportional to the area of the polygons resulted in a total of 62,920 points in the training set. The distribution of samples is presented in <a href="#fig-ground_data">Figure&nbsp;<span>18.4</span></a> .</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-ground_data" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/ground_data_distribution.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.4: Distribution of training data by Level 3 class for MZ4 (Source: National Institute of Statistics, Chile).</figcaption></figure>
</div>
</div>
</div>
</section><section id="generation-of-land-use-and-land-cover-maps-using-time-series-of-satellite-images" class="level3" data-number="18.2.4"><h3 data-number="18.2.4" class="anchored" data-anchor-id="generation-of-land-use-and-land-cover-maps-using-time-series-of-satellite-images">
<span class="header-section-number">18.2.4</span> Generation of land use and land cover maps using time series of satellite images</h3>
<p><span class="citation" data-cites="Simoes2021"><a href="#ref-Simoes2021" role="doc-biblioref">[24]</a></span> developed an open-source R package called <code>sits</code>, which enables the analysis of time series of satellite images for land use and land cover classification. This analysis is performed through machine learning and follows an approach they have termed <em>“time first–space later.”</em> The main contribution of this package is that it provides a complete workflow for soil classification using large Earth Observation datasets. Users can create data cubes from cloud-based image providers, retrieve time series from these cubes, and improve the quality of training data. Moreover, various machine learning and deep learning methods are supported. Additionally, spatial smoothing methods remove outliers from classifications <span class="citation" data-cites="Camara2024b"><a href="#ref-Camara2024b" role="doc-biblioref">[25]</a></span>, and best-practice accuracy techniques ensure realistic assessments <span class="citation" data-cites="Olofsson2013 Olofsson2014"><a href="#ref-Olofsson2013" role="doc-biblioref">[26]</a>, <a href="#ref-Olofsson2014" role="doc-biblioref">[27]</a></span>.</p>
<p>The <code>sits</code> package introduces innovations in image catalog access for cloud computing services and deep learning algorithms for time series analysis, which have been recently published and are unavailable in other R packages. It also includes new methods for quality control of training data <span class="citation" data-cites="Santos2021"><a href="#ref-Santos2021" role="doc-biblioref">[28]</a></span>. Parallel processing methods specifically designed for data cubes ensure efficient performance. Sits provides tools for analysis, visualization, and classification of time series of satellite images (supporting the full classification cycle), offering features that surpass those of existing R packages. It proposes a standardized workflow for users to perform pixel-based classification.</p>
<p>The package provides key tools for generating maps with potential applications in official statistics, aligning with FAO recommendations for accuracy assessment and area estimation in land use and land cover maps <span class="citation" data-cites="Finegold2016"><a href="#ref-Finegold2016" role="doc-biblioref">[29]</a></span>. It allows accuracy to be measured through confusion matrices, area estimates, and confidence intervals that reflect the precision of these estimates.</p>
<section id="the-sits-workflow" class="level4 unnumbered"><h4 class="unnumbered anchored" data-anchor-id="the-sits-workflow">The <em>sits</em> workflow</h4>
<p>The package provides a comprehensive set of tools for the analysis, visualization, and classification of satellite image time series. It supports a standardized workflow for pixel-based classification, in which each step corresponds to a dedicated function in the <code>sits</code> API, with well-defined inputs and outputs, as illustrated in <a href="#fig-sits">Figure&nbsp;<span>18.5</span></a>. The API is designed to be intuitive and easy to use, with sensible default parameters and user-friendly behavior. As a result, users can achieve reliable results without requiring deep expertise in machine learning or parallel processing (All the steps and example scripts could be found in <a href="https://e-sensing.github.io/sitsbook/introduction.html#how-sits-works">sitsbook</a>).</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-sits" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/sits_workflow.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.5: SITS API workflow for LULC classification (Source: adapted from sitsbook)</figcaption></figure>
</div>
</div>
</div>
</section></section></section><section id="configuration-to-run-the-example-in-this-chapter" class="level2 unnumbered"><h2 class="unnumbered anchored" data-anchor-id="configuration-to-run-the-example-in-this-chapter">Configuration to run the example in this chapter</h2>
<div class="cell">
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># load packages "tibble", "sf" and "sits"</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://tibble.tidyverse.org/">tibble</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/e-sensing/sits/">sits</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://r-spatial.github.io/sf/">sf</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://rspatial.org/">terra</a></span><span class="op">)</span></span>
<span></span>
<span><span class="co"># set dir if it does not exist</span></span>
<span><span class="va">dir_19HBA</span> <span class="op">&lt;-</span> <span class="st">"data/ct_chile"</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html">dir.create</a></span><span class="op">(</span><span class="va">dir_19HBA</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># set dir for results</span></span>
<span><span class="va">dir_19HBA_out</span>     <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_19HBA</span>, <span class="st">"/Out"</span><span class="op">)</span></span>
<span><span class="va">dir_19HBA_cube</span>    <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_19HBA</span>, <span class="st">"/Cube"</span><span class="op">)</span></span>
<span><span class="va">dir_19HBA_cubeS2</span>  <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_19HBA_cube</span>, <span class="st">"/S2_Cube"</span><span class="op">)</span></span>
<span><span class="va">dir_19HBA_cubeS2reg</span>  <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_19HBA_cube</span>, <span class="st">"/S2_Cube/reg"</span><span class="op">)</span></span>
<span><span class="va">dir_19HBA_cubeDEM</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_19HBA_cube</span>, <span class="st">"/DEM_Cube"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html">dir.create</a></span><span class="op">(</span><span class="va">dir_19HBA_out</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html">dir.create</a></span><span class="op">(</span><span class="va">dir_19HBA_cube</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html">dir.create</a></span><span class="op">(</span><span class="va">dir_19HBA_cubeS2</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html">dir.create</a></span><span class="op">(</span><span class="va">dir_19HBA_cubeS2reg</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html">dir.create</a></span><span class="op">(</span><span class="va">dir_19HBA_cubeDEM</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> </span>
<span></span>
<span><span class="co"># Set the color palette</span></span>
<span><span class="va">cl_tbl_eng</span> <span class="op">&lt;-</span> <span class="fu">tibble</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/tibble.html">tibble</a></span><span class="op">(</span>name <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/character.html">character</a></span><span class="op">(</span><span class="op">)</span>, color <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/character.html">character</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">cl_tbl_eng</span> <span class="op">&lt;-</span> <span class="va">cl_tbl_eng</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Sand_dunes"</span>,          color <span class="op">=</span> <span class="st">"#bababa"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Fallow"</span>,              color <span class="op">=</span> <span class="st">"#7D4A0C"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Deciduous_forest"</span>,    color <span class="op">=</span> <span class="st">"#228710"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Perennial_forest"</span>,    color <span class="op">=</span> <span class="st">"#004529"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Winter_crops"</span>,        color <span class="op">=</span> <span class="st">"#ffff99"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Spring_crops"</span>,        color <span class="op">=</span> <span class="st">"#ffd92f"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Successive_crops"</span>,    color <span class="op">=</span> <span class="st">"#FFA500"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Deciduous_fruit_tree"</span>,color <span class="op">=</span> <span class="st">"#df65b0"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Perennial_fruit_tree"</span>,color <span class="op">=</span> <span class="st">"#ce1256"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Wetlands"</span>,            color <span class="op">=</span> <span class="st">"#93dfe6"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Shrublands"</span>,          color <span class="op">=</span> <span class="st">"#8B864E"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Snow_glaciers"</span>,       color <span class="op">=</span> <span class="st">"#00BFFF"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Annual_pasture"</span>,      color <span class="op">=</span> <span class="st">"#6959CD"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Perennial_pasture"</span>,   color <span class="op">=</span> <span class="st">"#B452CD"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Mature_plantation"</span>,   color <span class="op">=</span> <span class="st">"#7CFC00"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Clear_cut_plantation"</span>,color <span class="op">=</span> <span class="st">"#bf812d"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Young_plantation"</span>,    color <span class="op">=</span> <span class="st">"#0ecf65"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Mountain_grasslands"</span>, color <span class="op">=</span> <span class="st">"#35978f"</span><span class="op">)</span><span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Grasslands"</span>,          color <span class="op">=</span> <span class="st">"#01665e"</span><span class="op">)</span><span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Bare_soils"</span>,          color <span class="op">=</span> <span class="st">"#e0e0e0"</span><span class="op">)</span><span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Water_bodies"</span>,        color <span class="op">=</span> <span class="st">"#0C3BD4"</span><span class="op">)</span><span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Artificial_surfaces"</span>, color <span class="op">=</span> <span class="st">"#fa0000"</span><span class="op">)</span><span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"No_Class"</span>,            color <span class="op">=</span> <span class="st">"white"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"No_Data"</span>,             color <span class="op">=</span> <span class="st">"white"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"No_Samples"</span>,          color <span class="op">=</span> <span class="st">"white"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Load the color table into `sits`</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_colors_set.html">sits_colors_set</a></span><span class="op">(</span><span class="va">cl_tbl_eng</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>In this example, we replicated the land use and land cover classification results developed for the entire study area but focused on a specific region to ensure the reproducibility of the process. To illustrate the workflow, we selected a small region covering tile 19HBA, which contains a high concentration of agricultural land (<a href="#fig-map">Figure&nbsp;<span>18.1</span></a>). The classification covers the period from May 2020 to May 2021, aligned with the agricultural calendar, and uses a temporal resolution of 16 days throughout the analysis. It is important to note that the workflow includes stochastic processes, such as model training or sampling steps, which may lead to slight variations in results across different runs, even when using the same input data and parameters.&nbsp;</p>
</section><section id="study-area-data" class="level2" data-number="18.3"><h2 data-number="18.3" class="anchored" data-anchor-id="study-area-data">
<span class="header-section-number">18.3</span> Study area data</h2>
<p>We begin by loading the spatial datasets for this study. The first file contains the ground-sampled points used as reference data, and the second defines the ROI that frames our analysis. Both are read with <code>sf</code> to retain their geometry and attributes for later spatial operations.</p>
<div class="cell">
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Read the samples and roi</span></span>
<span><span class="va">points_roi</span>  <span class="op">&lt;-</span> <span class="fu"><a href="https://r-spatial.github.io/sf/reference/st_read.html">st_read</a></span><span class="op">(</span><span class="st">"data/ct_chile/Samples/ground_data_19HBA_Chile_eng.shp"</span><span class="op">)</span></span>
<span><span class="va">roi_test</span>    <span class="op">&lt;-</span> <span class="fu"><a href="https://r-spatial.github.io/sf/reference/st_read.html">st_read</a></span><span class="op">(</span><span class="st">"data/ct_chile/ROI/ROI_19HBA.shp"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="earth-observation-data-cubes" class="level2" data-number="18.4"><h2 data-number="18.4" class="anchored" data-anchor-id="earth-observation-data-cubes">
<span class="header-section-number">18.4</span> Earth observation data cubes</h2>
<section id="accessing-sentinel-22a-images" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="accessing-sentinel-22a-images">Accessing SENTINEL-2/2A images</h3>
<p>For the case of MZ4 we build an irregular data cube with 14 tiles that cover the entire area (<code>18HXF, 18HYG, 18HXE, 18HYF, 18HYE, 18HYD, 19HBB, 19HBA, 19HBV, 19HBU, 19HCB, 19HCA, 19HCV, 19HCU</code>) and request 11 bands (<code>B02, B03, B04, B05, B06, B07, B08, B8A, B11, B12, CLOUD</code>). For this example we only request 3 bands plus the Cloud band.</p>
<div class="cell">
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Create an irregular data cube covering a small area of the MZ4</span></span>
<span><span class="co"># Access the Sentinel through the MPC</span></span>
<span><span class="va">sent_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_cube.html">sits_cube</a></span><span class="op">(</span></span>
<span>  source         <span class="op">=</span> <span class="st">"MPC"</span>,                           </span>
<span>  collection     <span class="op">=</span> <span class="st">"SENTINEL-2-L2A"</span>,                </span>
<span>  tiles          <span class="op">=</span> <span class="st">"19HBA"</span>,                          </span>
<span>  bands          <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"B04"</span>, <span class="st">"B08"</span>, <span class="st">"B11"</span>, <span class="st">"CLOUD"</span><span class="op">)</span>, </span>
<span>  start_date     <span class="op">=</span> <span class="st">"2020-05-01"</span>,                    </span>
<span>  end_date       <span class="op">=</span> <span class="st">"2021-05-30"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>The generated data cube is irregular, so it must be regularized before proceeding with the analysis. As recommended in the <a href="https://e-sensing.github.io/sitsbook/dc_regularize.html">sitsbook</a> documentation, we first download the images locally prior to applying the regularization step. While this is not strictly required, it is considered a best practice. However, if local storage is limited, regularization can be performed directly on the remote data source.</p>
<div class="cell">
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Copy images locally</span></span>
<span><span class="va">sent_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_cube_copy.html">sits_cube_copy</a></span><span class="op">(</span></span>
<span>  cube       <span class="op">=</span> <span class="va">sent_19HBA</span>,</span>
<span>  multicores <span class="op">=</span> <span class="fl">5</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="va">dir_19HBA_cubeS2</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>The <code>sits</code> package supports access to both local and remote image collections. Next step demonstrate how to read locally stored images after copying them from the remote source. The only distinction between accessing local versus remote data lies in the <code>data_dir</code> parameter, which indicates the directory path where the images are stored locally.</p>
<div class="cell">
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Access local data cube</span></span>
<span><span class="va">sent_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_cube.html">sits_cube</a></span><span class="op">(</span></span>
<span>    source     <span class="op">=</span> <span class="st">"MPC"</span>,</span>
<span>    collection <span class="op">=</span> <span class="st">"SENTINEL-2-L2A"</span>,</span>
<span>    data_dir   <span class="op">=</span> <span class="va">dir_19HBA_cubeS2</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>After creating the local data cube, we proceed with the regularization step using the <code><a href="https://rdrr.io/pkg/sits/man/sits_regularize.html">sits_regularize()</a></code> function. In this case, the entire timeline of the irregular data cube is regularized, assuming it shares the same timeline as the ground truth data. A temporal resolution of 16 days was defined for the regularization process.</p>
<div class="cell">
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Regularize the cube to 16 day intervals </span></span>
<span><span class="va">sent_19HBA_reg</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_regularize.html">sits_regularize</a></span><span class="op">(</span></span>
<span>  cube       <span class="op">=</span> <span class="va">sent_19HBA</span>,                  </span>
<span>  period     <span class="op">=</span> <span class="st">"P16D"</span>,                         </span>
<span>  output_dir <span class="op">=</span> <span class="va">dir_19HBA_cubeS2reg</span>,  </span>
<span>  res        <span class="op">=</span> <span class="fl">10</span>,                            </span>
<span>  multicores <span class="op">=</span> <span class="fl">8</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Plot</span></span>
<span><span class="va">sent_19HBA_reg</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span>red <span class="op">=</span> <span class="st">"B08"</span>, blue <span class="op">=</span> <span class="st">"B04"</span>, green <span class="op">=</span> <span class="st">"B11"</span>,</span>
<span>       dates <span class="op">=</span> <span class="st">"2020-12-13"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-reg_19hba" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/reg_19HBA_sentinel2.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.6: Regularized image for Sentinel-2 tile 19HBA</figcaption></figure>
</div>
</div>
</div>
</section><section id="copernicus-dem-30-meter-images" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="copernicus-dem-30-meter-images">Copernicus DEM 30 meter images</h3>
<p>In regions with significant topographic, soil, or climatic variability, integrating multitemporal satellite data with elevation information, such as Digital Elevation Models (DEM), can improve land use and land cover classification. Elevation adds a third dimension that helps differentiate classes influenced by altitude, such as lowland vs.&nbsp;highland forests.</p>
<p>Given that both MZ4 and the region of interest (ROI) exhibit considerable topographic variability, with a steep altitudinal gradient (<a href="#fig-map">Figure&nbsp;<span>18.1</span></a>), it becomes relevant to integrate elevation data into the analysis. In mountainous countries like Chile, where elevation strongly influences climatic and soil conditions, this altitudinal variability plays a key role in shaping the spatial distribution of land use and vegetation cover. In our example, we integrate a DEM into a data cube (this agricultural region in Chile covers the Andes, where steep elevation gradients justify its inclusion). We use the Copernicus Global DEM-30, selecting the area covered by the 19HBA tile. For MZ4, we selected the entire ROI within the study area.</p>
<div class="cell">
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># DEM irregular cube </span></span>
<span><span class="va">dem_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_cube.html">sits_cube</a></span><span class="op">(</span></span>
<span>  source      <span class="op">=</span> <span class="st">"MPC"</span>,             </span>
<span>  collection  <span class="op">=</span> <span class="st">"COP-DEM-GLO-30"</span>,  </span>
<span>  tiles       <span class="op">=</span> <span class="st">"19HBA"</span>,          </span>
<span>  bands       <span class="op">=</span> <span class="st">"ELEVATION"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>After obtaining the data cube for the selected tile, the next step is to regularize it using the <code><a href="https://rdrr.io/pkg/sits/man/sits_regularize.html">sits_regularize()</a></code> function. This function generates a regularized data cube that exactly matches the spatial and temporal extent of the selected tile.</p>
<div class="cell">
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Regularizing DEM cube</span></span>
<span><span class="va">dem_19HBA_reg</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_regularize.html">sits_regularize</a></span><span class="op">(</span></span>
<span>  cube       <span class="op">=</span> <span class="va">dem_19HBA</span>,</span>
<span>  res        <span class="op">=</span> <span class="fl">10</span>,</span>
<span>  bands      <span class="op">=</span> <span class="st">"ELEVATION"</span>,</span>
<span>  tiles      <span class="op">=</span> <span class="st">"19HBA"</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="va">dir_19HBA_cubeDEM</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># plot the DEM </span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">dem_19HBA_reg</span>, band <span class="op">=</span> <span class="st">"ELEVATION"</span>, palette <span class="op">=</span> <span class="st">"RdYlBu"</span>, rev <span class="op">=</span> <span class="cn">T</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-dem" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/dem_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.7: Copernicus DEM-30 covering tile 19HBA.</figcaption></figure>
</div>
</div>
</div>
</section><section id="computing-vegetation-indexes" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="computing-vegetation-indexes">Computing vegetation indexes</h3>
<p>The <code>sits</code> package provides powerful tools for performing operations directly on data cubes, enabling users to manipulate and extract information from multitemporal satellite imagery. One of these key functions is <code><a href="https://rdrr.io/pkg/sits/man/sits_apply.html">sits_apply()</a></code>, which allows the computation of custom indices using R expressions. In the step below, we illustrate its use by calculating the NDVI index. For MZ4, we apply this function to compute NDVI, EVI, and NDWI indices, and we also derive the <em>slope</em> from the DEM using the <code>terra</code> package, enhancing the feature set with topographic information.</p>
<div class="cell">
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Calculate NDVI index using bands B08 and B04</span></span>
<span><span class="va">sent_19HBA_reg</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_apply.html">sits_apply</a></span><span class="op">(</span><span class="va">sent_19HBA_reg</span>,</span>
<span>                          NDVI <span class="op">=</span> <span class="op">(</span><span class="op">(</span><span class="va">B08</span> <span class="op">-</span> <span class="va">B04</span><span class="op">)</span> <span class="op">/</span> <span class="op">(</span><span class="va">B08</span> <span class="op">+</span> <span class="va">B04</span><span class="op">)</span><span class="op">)</span>,</span>
<span>                          memsize <span class="op">=</span> <span class="fl">32</span>,</span>
<span>                          multicores <span class="op">=</span> <span class="fl">8</span>,</span>
<span>                          output_dir <span class="op">=</span> <span class="va">dir_19HBA_cubeS2</span>,    </span>
<span>                          progress <span class="op">=</span> <span class="cn">T</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Visualize NDVI</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">sent_19HBA_reg</span>, band <span class="op">=</span> <span class="st">"NDVI"</span>, dates <span class="op">=</span> <span class="st">"2021-01-14"</span>,palette <span class="op">=</span> <span class="st">"RdYlGn"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-ndvi" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/ndvi_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.8: NDVI using bands B08 and B04 of Sentinel-2 for 19HBA tile.</figcaption></figure>
</div>
</div>
</div>
</section><section id="combining-multitemporal-data-cubes-with-dems" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="combining-multitemporal-data-cubes-with-dems">Combining multitemporal data cubes with DEMs</h3>
<p>As stated in the <code>sitsbook</code>, an alternative for incorporating elevation data is to merge the image data cube with the DEM using the <code><a href="https://rdrr.io/pkg/sits/man/sits_merge.html">sits_merge()</a></code> function, treating the DEM as an additional band. This replicates the DEM across all time steps, creating an artificial time series. It can be particularly beneficial in areas where elevation strongly influences land cover patterns.</p>
<div class="cell">
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Merging multitemporal data cubes with DEM</span></span>
<span><span class="co"># In the case of Chile, the DEM should be used as a time series band</span></span>
<span><span class="va">cube_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_merge.html">sits_merge</a></span><span class="op">(</span><span class="va">sent_19HBA_reg</span>, <span class="va">dem_19HBA_reg</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section></section><section id="satellite-image-time-series" class="level2" data-number="18.5"><h2 data-number="18.5" class="anchored" data-anchor-id="satellite-image-time-series">
<span class="header-section-number">18.5</span> Satellite image time series</h2>
<section id="getting-time-series-data-from-data-cubes" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="getting-time-series-data-from-data-cubes">Getting time series data from data cubes</h3>
<p>After creating the data cube, the next step is to extract the time series for the ground truth data. In this case we use an <code>sf</code> object with <code>POINT</code> geometry as <code>samples</code> parameter (See <a href="#sec-training-data"><span>Section&nbsp;18.2.3</span></a>). In this case, all samples share the same dates and correspond to those of the data cube.</p>
<div class="cell">
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Read the points from the cube and produce a tibble with time series</span></span>
<span><span class="va">samples_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_get_data.html">sits_get_data</a></span><span class="op">(</span></span>
<span>  cube       <span class="op">=</span> <span class="va">cube_19HBA</span>,            </span>
<span>  samples    <span class="op">=</span> <span class="va">points_roi</span>,              </span>
<span>  labet_attr <span class="op">=</span> <span class="st">"label"</span>,                </span>
<span>  pol_id     <span class="op">=</span> <span class="st">"id"</span>,                   </span>
<span>  progress   <span class="op">=</span> <span class="cn">T</span>,</span>
<span>  multicores <span class="op">=</span> <span class="fl">8</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Show summary of samples</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/summary.html">summary</a></span><span class="op">(</span><span class="va">samples_19HBA</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 22 × 3
   label                count    prop
   &lt;chr&gt;                &lt;int&gt;   &lt;dbl&gt;
 1 Annual_pasture         699 0.0328 
 2 Artificial_surfaces   1622 0.0761 
 3 Bare_soils             312 0.0146 
 4 Clear_cut_plantation  1095 0.0514 
 5 Deciduous_forest      1312 0.0616 
 6 Deciduous_fruit_tree  2653 0.124  
 7 Fallow                1097 0.0515 
 8 Grasslands             372 0.0175 
 9 Mature_plantation     2099 0.0985 
10 Mountain_grasslands     92 0.00432
11 Perennial_forest       169 0.00793
12 Perennial_fruit_tree  1203 0.0565 
13 Perennial_pasture     1423 0.0668 
14 Sand_dunes             145 0.00680
15 Shrublands            1021 0.0479 
16 Snow_glaciers          123 0.00577
17 Spring_crops          1416 0.0664 
18 Successive_crops       226 0.0106 
19 Water_bodies          1114 0.0523 
20 Wetlands                81 0.00380
21 Winter_crops          1458 0.0684 
22 Young_plantation      1578 0.0740 </code></pre>
</div>
</div>
<p>For the region of interest in this example, we obtained a time series of 21,291 samples. For the entire MZ4, the number of samples is 62,823 (some points could not be retrieved).</p>
<p>When working with extensive time series data, it is helpful to generate a single plot that summarizes the key temporal dynamics of each class. The function <code><a href="https://rdrr.io/pkg/sits/man/sits_patterns.html">sits_patterns()</a></code> predict an idealized approximation of the time series associated with each class across bands. The resulting patterns can be visualized using the <code><a href="https://rspatial.github.io/terra/reference/plot.html">plot()</a></code> function.</p>
<p>For better visualization, the 22 Level 3 classes available in this area of interest were grouped into four main categories: <em>Agricultural Lands,</em> which includes the eight agricultural Level 3 classes; <em>Forest Plantation</em>, which consists of the three forest-related Level 3 categories; <em>Native Vegetation</em>, comprising six Level 3 classes associated with areas covered by natural vegetation; and <em>Non-Vegetated Areas</em>, which includes five Level 3 categories, both natural and artificial, that lack vegetation cover.</p>
<div class="cell">
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Estimate the patterns for each class and plot them</span></span>
<span><span class="va">samples_19HBA</span> <span class="op">|&gt;</span> </span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_select.html">sits_select</a></span><span class="op">(</span>bands <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"B04"</span>, <span class="st">"B08"</span>, <span class="st">"B11"</span>, <span class="st">"NDVI"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_patterns.html">sits_patterns</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>  <span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" aria-current="page">Agricultural lands</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">Forest plantation</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">Native vegetation</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false">Non vegetated</a></li>
</ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-patt_agric" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/patterns_agric_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.9: Agricultural Land patterns in 19HBA sample data.</figcaption></figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-patt_forest" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/patterns_plant_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.10: Forest plantations patterns in 19HBA sample data.</figcaption></figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-patt_nat" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/patterns_native_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.11: Natural vegetation patterns in 19HBA sample data.</figcaption></figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-patt_non_veg" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/patterns_no_veg_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.12: Non vegetated areas patterns in 19HBA sample data.</figcaption></figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>The curves reveal distinctive patterns across different land cover groups. In the case of <em>Agricultural lands</em>, most classes share a phenological behavior characterized by NDVI peaks and declines throughout the year, corresponding to the periods of greatest vegetative vigor for each crop type. An exception is the <code>Fallow</code> class, which maintains consistently low NDVI values year-round, mainly due to the absence of active cultivation. In contrast, <em>Perennial crops</em> (pastures and fruit trees) classes exhibit less spectral variation throughout the year, reflecting the relatively stable vegetation cover typical of these crop types.</p>
<p>In the case of <em>Forest Plantation</em> classes, the NDVI curves reflect the distinctive characteristics of each category. The <code>Clear-Cut Plantation</code> class shows low vegetative vigor, especially during the summer and autumn months, with a slight increase toward the end of winter. This pattern is explained by the fact that these areas were previously forest plantations that have been harvested, leaving bare soil where grasses begin to emerge after the rainy season. In contrast, the <em>Mature Plantation</em> and <em>Young Plantation</em> classes maintain more stable NDVI values throughout the year, with higher values observed in mature stands. This behavior corresponds to the nature of these plantations, which are composed of fast-growing, evergreen species that provide continuous vegetation cover.</p>
<p>The classes grouped under <em>Native vegetation</em> show vegetation vigor behavior according to the characteristics of each class. For the <code>Deciduous_forest</code> class, a marked decrease in NDVI values is observed in the winter season due to the natural loss of foliage. In contrast, the <code>Perennial_forest</code> maintains high and constant values throughout the analysis season. For <code>Grasslands</code>, an increase in vigor is observed in late winter and early spring, then a marked decrease during summer and autumn and early winter, when these natural grasslands have low vegetation vigor. The <code>Shrublands</code> class shows a slight NDVI increase in late winter and early spring, followed by a moderate decline. This pattern reflects the seasonal variation in vegetation vigor, influenced by sparse perennial shrubs and herbaceous understory responding to wet and dry periods. Finally, in the case of <code>Wetlands,</code> these areas exhibit a brief period of low vegetation vigor, shifting to negative NDVI values corresponding to the months in which these natural areas are covered by a body of water, as part of their annual variation in conjunction with the wet and dry months of the year.</p>
<p>Finally, for the classes grouped under <em>Non-Vegetated Areas</em>, all exhibit very low NDVI values, lower than those observed in the other plotted spectral bands, reflecting the absence of vegetation in these land cover and land use categories. The temporal variation across the four spectral bands is minimal, particularly for classes such as <code>Artificial_surfaces</code>, <code>Sand_dunes,</code> and <code>Bare_soils</code>, which show little seasonal change. In contrast, <code>Snow_glaciers</code> and <code>Water_bodies</code> exhibit slight variations across the four spectral bands, mainly in response to minor annual fluctuations in their natural behavior.</p>
<p>These temporal and spectral variations suggest that classification algorithms can effectively exploit phenological differences to distinguish between land cover classes, even when some exhibit similar spectral signatures.</p>
<section id="self-organized-maps-for-sample-quality-control" class="level4 unnumbered"><h4 class="unnumbered anchored" data-anchor-id="self-organized-maps-for-sample-quality-control">Self-organized maps for sample quality control</h4>
<p>Since the data sources used were not originally designed for Land Use and Land Cover (LULC) mapping, it is essential to ensure the quality of training samples before using them in classification models. High-quality samples are critical for achieving accurate results with remote sensing and machine learning techniques, regardless of the algorithm applied <span class="citation" data-cites="Santos2021 deSimone2025"><a href="#ref-Santos2021" role="doc-biblioref">[28]</a>, <a href="#ref-deSimone2025" role="doc-biblioref">[30]</a></span>. Both the quantity and quality of training data significantly influence model performance. Well-labeled and representative datasets tend to yield better outcomes, while noisy or misclassified samples can degrade classification accuracy. Therefore, implementing preprocessing methods to detect and remove low-quality or mislabeled samples is a key step in improving model reliability and reducing classification errors <span class="citation" data-cites="Simoes2021 Santos2021"><a href="#ref-Simoes2021" role="doc-biblioref">[24]</a>, <a href="#ref-Santos2021" role="doc-biblioref">[28]</a></span>.</p>
<p>The <code>sits</code> package provides a clustering technique based on Self-Organizing Maps (SOM) as an alternative to hierarchical clustering methods for quality control of training samples. SOM is a dimensionality reduction technique that projects high-dimensional data onto a two-dimensional map while preserving the topological relationships between data patterns. This property facilitates the visualization and assessment of the internal consistency of the samples, helping to identify outliers, mislabeled classes, or regions with high spectral-temporal heterogeneity that could negatively impact the performance of the classification model.</p>
<p>For model parsimony, only a subset of bands from the time series is selected, those that best capture the relevant temporal and spectral variability. This reduced set of information is then used for quality assessment, as well as for the subsequent training and classification processes, ensuring computational efficiency without compromising model performance. For the MZ4 we selected the following bands: <code>NDVI, EVI, NDWI, B8A, B11, B12, B07, B05, B02, ELEVATION, SLOPE</code>. In this example we use all the presents bands, which are <code>NDVI, B04, B08, B11</code>, and <code>ELEVATION</code>.</p>
<section id="creating-the-som-map" class="level5 unnumbered"><h5 class="unnumbered anchored" data-anchor-id="creating-the-som-map">Creating the SOM map</h5>
<p>To perform the quality assessment based on SOM, the first step is to run <code><a href="https://rdrr.io/pkg/sits/man/sits_som_map.html">sits_som_map()</a></code>. The grid size is defined by the sample size, according the suggestions by <span class="citation" data-cites="Vesanto2000"><a href="#ref-Vesanto2000" role="doc-biblioref">[31]</a></span>, as implemented in <span class="citation" data-cites="Santos2021"><a href="#ref-Santos2021" role="doc-biblioref">[28]</a></span> and <span class="citation" data-cites="deSimone2025"><a href="#ref-deSimone2025" role="doc-biblioref">[30]</a></span>.</p>
<p>For the implementation of the other parameters see the <a href="https://e-sensing.github.io/sitsbook/ts_som.html">“Self-Organized maps for sample quality control”</a> chapter in <code>sitsbook</code> and <span class="citation" data-cites="Santos2021"><a href="#ref-Santos2021" role="doc-biblioref">[28]</a></span>.</p>
<div class="cell">
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Determine grid size</span></span>
<span><span class="va">n_samples</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html">length</a></span><span class="op">(</span><span class="va">samples_19HBA</span><span class="op">$</span><span class="va">longitude</span><span class="op">)</span></span>
<span><span class="va">min_grid</span>  <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Round.html">floor</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">sqrt</a></span><span class="op">(</span><span class="fl">5</span> <span class="op">*</span> <span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">sqrt</a></span><span class="op">(</span><span class="va">n_samples</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">-</span> <span class="fl">2</span> </span>
<span></span>
<span><span class="co">#  Clustering time series using SOM</span></span>
<span><span class="va">som_map</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_som_map.html">sits_som_map</a></span><span class="op">(</span>data      <span class="op">=</span> <span class="va">samples_19HBA</span>,    </span>
<span>                        grid_xdim <span class="op">=</span> <span class="va">min_grid</span>,         </span>
<span>                        grid_ydim <span class="op">=</span> <span class="va">min_grid</span>,          </span>
<span>                        alpha     <span class="op">=</span> <span class="fl">0.05</span>,              </span>
<span>                        distance  <span class="op">=</span> <span class="st">"dtw"</span>,            </span>
<span>                        rlen      <span class="op">=</span> <span class="fl">50</span>,              </span>
<span>                        mode      <span class="op">=</span> <span class="st">"pbatch"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Plot SOM map</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">som_map</span>, band <span class="op">=</span> <span class="st">"NDVI"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-som_map_19HBA" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/som_map_19HBA.png" class="img-fluid figure-img" style="width:85.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.13: SOM map for the 19HBA tile samples.</figcaption></figure>
</div>
</div>
</div>
<p>The SOM map provides a general view of the spectral similarity among different land cover and land use classes, while also highlighting the separation between them. According to the legend, when a class appears as a compact grouping of a specific color, and that color does not appear scattered in other regions of the map, it indicates that the class exhibits high internal spectral consistency. This means that the samples share similar spectral behavior and are distinguishable from other classes. A clear example of this is the <code>Artificial_surfaces</code> class, which appears as a well-defined cluster in the lower-right section of the SOM map, suggesting strong differentiation from other categories. Conversely, when a class does not form a distinct cluster and its color is scattered across different areas of the map, this suggests low internal spectral consistency, meaning the samples within that class display varied spectral behaviors or overlap with other classes. This is the case for the <code>Annual_pastures</code> class, which appears highly fragmented and mixed with different classes, such as <code>Shrublands</code>. This dispersion indicates potential spectral confusion or high internal variability. In such cases, it may be advisable to reassess the consistency of the training samples or consider adjustments in the thematic class definitions.</p>
</section><section id="measuring-confusion-between-labels-using-som" class="level5 unnumbered"><h5 class="unnumbered anchored" data-anchor-id="measuring-confusion-between-labels-using-som">Measuring confusion between labels using SOM</h5>
<p>The next step in the SOM-based quality assessment involves analyzing label confusion. For this purpose, the <code>sits</code> package provides <code><a href="https://rdrr.io/pkg/sits/man/sits_som_evaluate_cluster.html">sits_som_evaluate_cluster()</a></code> function that groups the neurons of the SOM according to their majority label and generates a tibble with the results. Each group corresponds to a specific label, and the tibble shows the percentage of samples from each label present in each cluster.</p>
<p>In an ideal scenario, all samples within a cluster would belong to a single label, reflecting a clear and accurate classification. However, in practice, clusters often contain samples from different labels, indicating confusion in the classification. This information is key to identifying and quantifying the overlap between labels and assessing the quality of the training dataset. Sample-level label confusion can be visualized in a bar chart, which illustrates the initial confusion among the training sample labels.</p>
<div class="cell">
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Produce a tibble with a summary of the mixed labels</span></span>
<span><span class="va">som_eval</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_som_evaluate_cluster.html">sits_som_evaluate_cluster</a></span><span class="op">(</span><span class="va">som_map</span><span class="op">)</span> </span>
<span></span>
<span><span class="co"># Plot the confusion between clusters</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">som_eval</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-confusion_cluster_orig" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/confusion_cluster_orig_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.14: Confusion between classes as measured by SOM.</figcaption></figure>
</div>
</div>
</div>
<p>The confusion by cluster reveals that some classes are well separated, with a dominant color in their corresponding cluster, such as <code>Snow_glaciers</code>, <code>Water_bodies</code>, <code>Deciduous_forest</code>, <code>Winter_crops</code>, and <code>Clear_cut_plantation</code>, indicating clear and consistent spectral patterns.However, several classes show overlap with spectrally or contextually similar categories. This is the case for <code>Perennial_fruit_tree,</code> which displays dispersion toward plantation areas, suggesting challenges in separating perennial orchards from managed forest systems. There is also considerable confusion between <code>Grasslands</code> and <code>Annual_pastures</code>, and between <code>Bare_soils</code> and <code>Artificial_surfaces</code>.</p>
</section><section id="detecting-noisy-samples-using-som" class="level5 unnumbered"><h5 class="unnumbered anchored" data-anchor-id="detecting-noisy-samples-using-som">Detecting noisy samples using SOM</h5>
<p>The analysis based on SOM maps allows the identification and eventually removal of samples that may be confused with other classes, using the discrete probability distribution associated with each neuron, through the <code><a href="https://rdrr.io/pkg/sits/man/sits_som_clean_samples.html">sits_som_clean_samples()</a></code> function. This approach evaluates probabilities by considering the frequency of occurrence of samples within their corresponding cluster. For this study (both for the example and the full dataset), a probabilistic filter was applied by setting a threshold of 0.5 for both probabilities.</p>
<div class="cell">
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Detecting noisy samples using SOM</span></span>
<span><span class="va">all_eval</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_som_clean_samples.html">sits_som_clean_samples</a></span><span class="op">(</span></span>
<span>  som_map <span class="op">=</span> <span class="va">som_map</span>,</span>
<span>  prior_threshold <span class="op">=</span> <span class="fl">0.5</span>,</span>
<span>  posterior_threshold <span class="op">=</span> <span class="fl">0.5</span>,</span>
<span>  keep <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"clean"</span>, <span class="st">"analyze"</span>, <span class="st">"remove"</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Print the sample distribution based on evaluation</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">all_eval</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-noise_samples" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/class_noise_orig_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.15: Distribution of samples using SOM evaluation.</figcaption></figure>
</div>
</div>
</div>
<p>In the context of our quality analysis using SOM, the <code>sits</code> package provides an option to automatically remove samples labeled as <em>remove</em>, which are likely to contain high levels of noise. However, in our specific case, we chose not to apply this functionality, as removing these samples would further increase class imbalance, particularly affecting underrepresented classes, and compromise the integrity of the training dataset.</p>
<p>As a result of the analysis, 3,644 samples (17% of the dataset) were identified as potentially noisy and candidates for removal. However, their exclusion would lead to a further increase in class imbalance, with changes ranging from 0.26% to 12.5%<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a>.</p>
</section></section><section id="reducing-imbalances-in-training-samples" class="level4 unnumbered"><h4 class="unnumbered anchored" data-anchor-id="reducing-imbalances-in-training-samples">Reducing imbalances in training samples</h4>
<p>Imbalanced training data is a common issue in Earth observation analysis, occurring when the distribution of samples across class labels is uneven. In the dataset used for this example, the two most frequent classes (<code>Deciduous_fruit_tree</code> and <code>Mature_plantation</code><em>)</em> account for 22.4% of all sample units. In contrast, the two least frequent classes (<code>Wetlands</code> and <code>Mountain_grasslands</code>) represent only 0.8% of the dataset<a href="#fn3" class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a>.</p>
<p>Although this distribution reflects the actual land cover in the Maule and Ñuble Regions, the imbalance in sample representation is a problematic characteristic of a training dataset. Machine learning algorithms tend to perform better on classes with more training instances, while instances from minority classes are more likely to be misclassified.. Therefore, reducing sample imbalance can positively impact classification accuracy <span class="citation" data-cites="Johnson2019"><a href="#ref-Johnson2019" role="doc-biblioref">[32]</a></span>.</p>
<p>The <code><a href="https://rdrr.io/pkg/sits/man/sits_reduce_imbalance.html">sits_reduce_imbalance()</a></code> function addresses training set imbalance by oversampling minority classes and undersampling majority ones (through SOM maps; more details in <a href="https://e-sensing.github.io/sitsbook/ts_balance.html">sitsbook</a>). It uses the SMOTE method for oversampling, which generates synthetic samples by interpolating between pairs of nearest neighbors within minority class clusters.</p>
<div class="cell">
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Minimum and maximum number will depend on how many samples there are in the smallest and largest class</span></span>
<span><span class="va">balanced_samples</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_reduce_imbalance.html">sits_reduce_imbalance</a></span><span class="op">(</span></span>
<span>  samples         <span class="op">=</span> <span class="va">samples_19HBA</span>,   </span>
<span>  n_samples_over  <span class="op">=</span> <span class="fl">200</span>,     </span>
<span>  n_samples_under <span class="op">=</span> <span class="fl">800</span>,   </span>
<span>  multicores      <span class="op">=</span> <span class="fl">8</span><span class="op">)</span> </span>
<span></span>
<span><span class="co"># Show summary of balanced samples</span></span>
<span><span class="co"># Each label has between 7% and 1.7% of the full set</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/summary.html">summary</a></span><span class="op">(</span><span class="va">balanced_samples</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 22 × 3
   label                count   prop
   &lt;chr&gt;                &lt;int&gt;  &lt;dbl&gt;
 1 Annual_pasture         699 0.0641
 2 Artificial_surfaces    804 0.0737
 3 Bare_soils             312 0.0286
 4 Clear_cut_plantation   616 0.0565
 5 Deciduous_forest       784 0.0719
 6 Deciduous_fruit_tree   724 0.0664
 7 Fallow                 524 0.0480
 8 Grasslands             372 0.0341
 9 Mature_plantation      728 0.0667
10 Mountain_grasslands    200 0.0183
11 Perennial_forest       200 0.0183
12 Perennial_fruit_tree   676 0.0620
13 Perennial_pasture      556 0.0510
14 Sand_dunes             200 0.0183
15 Shrublands             612 0.0561
16 Snow_glaciers          200 0.0183
17 Spring_crops           468 0.0429
18 Successive_crops       226 0.0207
19 Water_bodies           568 0.0521
20 Wetlands               200 0.0183
21 Winter_crops           512 0.0469
22 Young_plantation       728 0.0667</code></pre>
</div>
</div>
<p>To evaluate the effect of imbalance reduction, as recommended by <code>sits</code> authors, we apply the SOM clustering technique introduced previously. We first generate the SOM using <code><a href="https://rdrr.io/pkg/sits/man/sits_som_map.html">sits_som_map()</a></code>, and then analyze the clustering results with <code>sits_som_evaluate_clusters()</code>.</p>
<div class="cell">
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Determine grid size for new SOM map</span></span>
<span><span class="va">n_samples</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html">length</a></span><span class="op">(</span><span class="va">balanced_samples</span><span class="op">$</span><span class="va">longitude</span><span class="op">)</span></span>
<span><span class="va">min_grid_size</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Round.html">floor</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">sqrt</a></span><span class="op">(</span><span class="fl">5</span> <span class="op">*</span> <span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">sqrt</a></span><span class="op">(</span><span class="va">n_samples</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">-</span> <span class="fl">2</span> </span>
<span></span>
<span><span class="co"># Clustering time series using SOM</span></span>
<span><span class="va">som_map_balanced</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_som_map.html">sits_som_map</a></span><span class="op">(</span>data <span class="op">=</span> <span class="va">balanced_samples</span>,</span>
<span>                                 grid_xdim <span class="op">=</span> <span class="va">min_grid_size</span>,</span>
<span>                                 grid_ydim <span class="op">=</span> <span class="va">min_grid_size</span>,</span>
<span>                                 alpha <span class="op">=</span> <span class="fl">0.05</span>,</span>
<span>                                 distance <span class="op">=</span> <span class="st">"dtw"</span>,</span>
<span>                                 rlen <span class="op">=</span> <span class="fl">50</span>, </span>
<span>                                 mode <span class="op">=</span> <span class="st">"pbatch"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Produce a tibble with a summary of the mixed labels</span></span>
<span><span class="va">som_eval_bal</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_som_evaluate_cluster.html">sits_som_evaluate_cluster</a></span><span class="op">(</span><span class="va">som_map_balanced</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Show the result</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">som_eval_bal</span><span class="op">)</span> </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-confusion_cluster_bal" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/confusion_cluster_smote_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.16: Confusion by cluster for the balanced dataset.</figcaption></figure>
</div>
</div>
</div>
<p>The confusion by cluster plot shows how well each class is grouped into a unique cluster after applying a combination of SMOTE oversampling and SOM-based undersampling. <code>Snow_glaciers</code>, <code>Wetlands</code>, <code>Mountain_grasslands</code>, and <code>Water_bodies</code> reach 97–100% purity in their assigned clusters, demonstrating a robust spectral distinction that was preserved or enhanced by the balancing process. Other classes, such as <code>Fallow</code>, <code>Clear_cut_plantation</code>, <code>Succesive_crops</code>, and <code>Perennial_forest</code>, also show high levels of purity (above 85%), confirming that the class balancing techniques helped consolidate their spectral representation and reduce overlap with other categories. However, some classes still present significant spectral confusion. For instance, <code>Perennial_fruit_tree</code>, <code>Shrublands</code>, <code>Grasslands</code>, and <code>Perennial_pasture</code> all show cluster mixtures, with multiple classes contributing to the same cluster. <code>Perennial_fruit_tree</code>, in particular, has a relatively low purity (68%), with noticeable overlaps with <code>Mature_plantation</code>, <code>Young_plantation</code>, and <code>Deciduous_forest</code>, suggesting persistent difficulty in distinguishing these vegetated classes.</p>
<p>The application of SMOTE and SOM-based balancing improved cluster purity for many classes, but some confusion remains, especially among spectrally or temporally similar vegetation types.</p>
</section></section></section><section id="training-a-machine-learning-model" class="level2" data-number="18.6"><h2 data-number="18.6" class="anchored" data-anchor-id="training-a-machine-learning-model">
<span class="header-section-number">18.6</span> Training a machine learning model</h2>
<p>The next step in creating a classified map involves training a model. We apply the <code><a href="https://rdrr.io/pkg/sits/man/sits_train.html">sits_train()</a></code> function to train a Random Forest (a CPU-based classification) classifier using the balanced sample data generated previously.</p>
<div class="cell">
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Train a random forest model</span></span>
<span><span class="va">rfor_model</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_train.html">sits_train</a></span><span class="op">(</span></span>
<span>  samples <span class="op">=</span> <span class="va">balanced_samples</span>,</span>
<span>  ml_method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_rfor.html">sits_rfor</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="sec-data-cube-classification" class="level2" data-number="18.7"><h2 data-number="18.7" class="anchored" data-anchor-id="sec-data-cube-classification">
<span class="header-section-number">18.7</span> Data cube classification</h2>
<p>After training the machine learning model, the process continues by applying this trained model to a satellite data cube. We use the <code><a href="https://rdrr.io/pkg/sits/man/sits_classify.html">sits_classify()</a></code> function, which requires a regular data cube and a trained model as input parameters; as additional information, we provide the region of interest (both for the example and the entire Macrozone) to generate a cube with cropped boundaries.</p>
<div class="cell">
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Classify the raster image</span></span>
<span><span class="va">probs_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_classify.html">sits_classify</a></span><span class="op">(</span></span>
<span>  data       <span class="op">=</span> <span class="va">cube_19HBA</span>,              </span>
<span>  ml_model   <span class="op">=</span> <span class="va">rfor_model</span>,   </span>
<span>  roi        <span class="op">=</span> <span class="va">roi_test</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="va">dir_19HBA_out</span>,</span>
<span>  version    <span class="op">=</span> <span class="st">"v1"</span>,</span>
<span>  multicores <span class="op">=</span> <span class="fl">8</span>,</span>
<span>  memsize    <span class="op">=</span> <span class="fl">16</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Plot the probabilities for one class</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">probs_19HBA</span>, labels <span class="op">=</span> <span class="st">"Deciduous_fruit_tree"</span>, palette <span class="op">=</span> <span class="st">"YlGn"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">19HBA full tile</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">Zoom to the tile</a></li>
</ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-probs_19hba_raw" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/probabilites_raw_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.17: Visualization of Deciduous fruit trees probabilities in 19HBA tile.</figcaption></figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-probs_19hba_raw_s" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/probabilites_raw_19HBA_small.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.18: Visualization of Deciduous fruit trees probabilities in 19HBA tile zoom.</figcaption></figure>
</div>
</div>
</div>
</div>
</div>
</div>
</section><section id="spatial-smoothing" class="level2" data-number="18.8"><h2 data-number="18.8" class="anchored" data-anchor-id="spatial-smoothing">
<span class="header-section-number">18.8</span> Spatial smoothing</h2>
<p>Spatial smoothing to the classification results. This step incorporates the spatial neighborhood of each pixel, helping to reduce isolated misclassifications, particularly in transitional zones.</p>
<p>The post-processing method is based on the Bayesian approach to data analysis, which interprets probability as a measure of belief in an event. To properly perform Bayesian smoothing, it is necessary to estimate the local logit variance for each class, which allows for the correct setting of the model’s hyperparameters. This estimation is carried out using the <code><a href="https://rdrr.io/pkg/sits/man/sits_variance.html">sits_variance()</a></code> function, which takes as input a probability data cube, a window size that defines the spatial neighborhood, and a fraction of pixels within that window to be used for variance calculation <span class="citation" data-cites="Camara2024b"><a href="#ref-Camara2024b" role="doc-biblioref">[25]</a></span>.</p>
<p>The distinct patterns of these classes are quantitatively assessed using the <code><a href="https://rspatial.github.io/terra/reference/summary.html">summary()</a></code> function. When applied to variance cubes, this function reports the logit variance values corresponding to the upper interquartile range, offering insights into the spatial variability within each class.</p>
<div class="cell">
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># calculate variance</span></span>
<span><span class="va">var_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_variance.html">sits_variance</a></span><span class="op">(</span></span>
<span>  cube           <span class="op">=</span> <span class="va">probs_19HBA</span>,</span>
<span>  window_size    <span class="op">=</span> <span class="fl">7</span>,</span>
<span>  neigh_fraction <span class="op">=</span> <span class="fl">0.50</span>,</span>
<span>  output_dir     <span class="op">=</span> <span class="va">dir_19HBA_out</span>,</span>
<span>  multicores     <span class="op">=</span> <span class="fl">4</span>,</span>
<span>  memsize        <span class="op">=</span> <span class="fl">16</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># get the summary of the logit variance</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/summary.html">summary</a></span><span class="op">(</span><span class="va">var_19HBA</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>     Annual_pasture Artificial_surfaces Bare_soils Clear_cut_plantation
75%          1.9425              3.5100     3.6600                 2.43
80%          3.1500              4.4000     4.5500                 3.56
85%          4.4800              5.2000     5.2000                 4.69
90%          5.4000              5.7700     5.6700                 5.55
95%          6.2400              6.5905     6.4305                 6.53
100%        17.7200             14.5500    14.1600                15.48
     Deciduous_forest Deciduous_fruit_tree Fallow Grasslands Mature_plantation
75%              3.34                 1.17  2.510       2.43            0.4300
80%              4.39                 2.43  3.822       3.66            0.5900
85%              5.20                 4.07  4.990       4.93            0.9500
90%              5.77                 5.35  5.730       5.66            2.6100
95%              6.57                 6.25  6.730       6.69            5.2005
100%            13.89                14.30 17.370      13.97           14.9900
     Mountain_grasslands Perennial_forest Perennial_fruit_tree
75%               0.0000             2.43               1.1425
80%               0.0220             3.66               2.4300
85%               0.8800             4.81               4.1015
90%               1.6900             5.60               5.2700
95%               3.4665             6.52               6.0600
100%             10.1200            19.82              14.4100
     Perennial_pasture Sand_dunes Shrublands Snow_glaciers Spring_crops
75%               1.76       1.69       0.41          0.00         3.90
80%               3.08       2.43       0.58          0.00         4.59
85%               4.43       3.66       0.91          0.00         5.20
90%               5.40       4.89       2.89          0.00         5.58
95%               6.13       5.60       5.40          0.00         6.03
100%             16.35      19.56      15.01         10.12        21.16
     Successive_crops Water_bodies Wetlands Winter_crops Young_plantation
75%              3.76         4.59     0.88         0.91             0.62
80%              4.59         5.14     1.69         2.43             0.96
85%              5.20         5.48     2.43         3.89             2.43
90%              5.58         5.77     3.90         5.16             4.59
95%              6.10         6.25     5.20         5.95             5.98
100%            11.08        21.66     9.68        10.66            22.73</code></pre>
</div>
</div>
<p>To remove the outliers and classification errors, we run a smoothing procedure with <code><a href="https://rdrr.io/pkg/sits/man/sits_smooth.html">sits_smooth()</a></code>. The parameters <code>window_size</code> and <code>neigh_fraction</code> control how many pixels in a neighborhood the Bayesian estimator will use to calculate the local statistics. The <code>smoothness</code> values for the classes are set as recommended in <span class="citation" data-cites="Camara2024b"><a href="#ref-Camara2024b" role="doc-biblioref">[25]</a></span> and <a href="https://e-sensing.github.io/sitsbook/cl_smoothing.html">“Bayesian smoothing for classification post-processing”</a> following the results of the summary of the logit variances.</p>
<div class="cell">
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Compute Bayesian smoothing</span></span>
<span><span class="va">smooth_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_smooth.html">sits_smooth</a></span><span class="op">(</span></span>
<span>  cube           <span class="op">=</span> <span class="va">probs_19HBA</span>,</span>
<span>  window_size    <span class="op">=</span> <span class="fl">7</span>,</span>
<span>  neigh_fraction <span class="op">=</span> <span class="fl">0.50</span>,</span>
<span>  smoothness <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Annual_pasture"</span>       <span class="op">=</span> <span class="fl">7</span>, <span class="st">"Perennial_fruit_tree"</span> <span class="op">=</span> <span class="fl">18</span>,</span>
<span>    <span class="st">"Artificial_surfaces"</span>  <span class="op">=</span> <span class="fl">7</span>, <span class="st">"Perennial_pasture"</span>    <span class="op">=</span> <span class="fl">7</span>,</span>
<span>    <span class="st">"Bare_soils"</span>           <span class="op">=</span> <span class="fl">15</span>,<span class="st">"Sand_dunes"</span>           <span class="op">=</span> <span class="fl">3</span>,</span>
<span>    <span class="st">"Clear_cut_plantation"</span> <span class="op">=</span> <span class="fl">14</span>,<span class="st">"Shrublands"</span>           <span class="op">=</span> <span class="fl">17</span>,</span>
<span>    <span class="st">"Deciduous_forest"</span>     <span class="op">=</span> <span class="fl">10</span>,<span class="st">"Snow_glaciers"</span>        <span class="op">=</span> <span class="fl">7</span>,</span>
<span>    <span class="st">"Deciduous_fruit_tree"</span> <span class="op">=</span> <span class="fl">15</span>,<span class="st">"Spring_crops"</span>         <span class="op">=</span> <span class="fl">5</span>,</span>
<span>    <span class="st">"Fallow"</span>               <span class="op">=</span> <span class="fl">5</span>, <span class="st">"Successive_crops"</span>     <span class="op">=</span> <span class="fl">5</span>,</span>
<span>    <span class="st">"Grasslands"</span>           <span class="op">=</span> <span class="fl">14</span>,<span class="st">"Water_bodies"</span>         <span class="op">=</span> <span class="fl">5</span>, </span>
<span>    <span class="st">"Mature_plantation"</span>    <span class="op">=</span> <span class="fl">1</span>, <span class="st">"Wetlands"</span>             <span class="op">=</span> <span class="fl">15</span>,</span>
<span>    <span class="st">"Mountain_grasslands"</span>  <span class="op">=</span> <span class="fl">9</span>,  <span class="st">"Winter_crops"</span>        <span class="op">=</span> <span class="fl">5</span>,</span>
<span>    <span class="st">"Perennial_forest"</span>     <span class="op">=</span> <span class="fl">10</span>,<span class="st">"Young_plantation"</span>     <span class="op">=</span> <span class="fl">15</span></span>
<span>    <span class="op">)</span>,</span>
<span>  multicores     <span class="op">=</span> <span class="fl">4</span>,</span>
<span>  memsize        <span class="op">=</span> <span class="fl">12</span>,</span>
<span>  output_dir     <span class="op">=</span> <span class="va">dir_19HBA_out</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Plot the result</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">smooth_19HBA</span>, labels <span class="op">=</span> <span class="st">"Deciduous_fruit_tree"</span>, palette <span class="op">=</span> <span class="st">"YlGn"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true">19HBA full tile</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false">Zoom to the tile</a></li>
</ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-smooth_prob" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/probabilites_smooth_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.19: Visualization of Deciduous fruit trees smooth probabilities in 19HBA tile.</figcaption></figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-smooth_prob_small" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/probabilites_smooth_19HBA_small.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.20: Visualization of Deciduous fruit trees smooth probabilities in 19HBA tile zoom.</figcaption></figure>
</div>
</div>
</div>
</div>
</div>
</div>
</section><section id="sec-labeling-a-probability-data-cube" class="level2" data-number="18.9"><h2 data-number="18.9" class="anchored" data-anchor-id="sec-labeling-a-probability-data-cube">
<span class="header-section-number">18.9</span> Labeling a probability data cube</h2>
<p>The final step consists of labeling the probability map by assigning class labels to the pixels with the highest probability values. This is done by selecting, for each pixel, the class with the maximum probability among all sample classes.</p>
<div class="cell">
<div class="sourceCode" id="cb25"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Generate the thematic map</span></span>
<span><span class="va">class_19HBA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_label_classification.html">sits_label_classification</a></span><span class="op">(</span></span>
<span>  cube <span class="op">=</span> <span class="va">smooth_19HBA</span>,</span>
<span>  multicores <span class="op">=</span> <span class="fl">4</span>,</span>
<span>  memsize <span class="op">=</span> <span class="fl">12</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="va">dir_19HBA_out</span>,</span>
<span>  version <span class="op">=</span> <span class="st">"v1"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">class_19HBA</span>,</span>
<span>     legend_text_size <span class="op">=</span> <span class="fl">0.7</span>, </span>
<span>     legend_position <span class="op">=</span> <span class="st">"outside"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-final_map" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/classified_smooth_19HBA.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.21: Final classification map after Bayesian smoothing for tile 19HBA.</figcaption></figure>
</div>
</div>
</div>
</section><section id="validation-and-accuracy-assessment-of-the-map" class="level2" data-number="18.10"><h2 data-number="18.10" class="anchored" data-anchor-id="validation-and-accuracy-assessment-of-the-map">
<span class="header-section-number">18.10</span> Validation and accuracy assessment of the map</h2>
<p>Up to this point, we have generated the classification for a small region to demonstrate how the <code>sits</code> package can be applied in a real-world scenario. For the validation step, we extended the classification to the entire MZ4 using reference samples designed according to the proposed sampling design.</p>
<p>Accuracy estimation with <code>sits</code> applies the best practices proposed by <span class="citation" data-cites="Olofsson2013"><a href="#ref-Olofsson2013" role="doc-biblioref">[26]</a></span>, integrating three key components: sampling design, reference data collection, and accuracy estimation. Specifically, the sampling design employs a random and stratified approach, ensuring representation of all land use and land cover classes in the sample, while the statistical methods used guarantee unbiased estimates. It also simplifies sample selection by class, facilitating an accurate assessment of map categories.</p>
<p>For MZ4, the sampling design was carried out using Level 2 of the hierarchical classification scheme. To do this, the classes originally present at Level 3 were reclassified using the <code><a href="https://rspatial.github.io/terra/reference/classify.html">classify()</a></code> function in the <code>terra</code> package, assigning them to their corresponding category in Level 2.</p>
<p>Validating the map at Level 2 represents a methodologically sound decision, as it balances the level of detail with the need for generalization, thus facilitating a robust and statistically significant evaluation of the model. Moreover, it fully meets the institutional objective of distinguishing between agricultural and non-agricultural lands Level 2 offers greater thematic disaggregation than Level 1, enabling the detection of relevant classification errors without incurring the excessive complexity of Level 3. This choice is crucial to ensure the quality and consistency of the classification, as it reduces the risk of errors due to spectral or semantic similarity between closely related classes, such as Temporary Crops and Permanent Crops.</p>
<p>Several studies have shown that with increasing class counts (such as at more disaggregated levels), thematic accuracy tends to decrease, and confounding errors increase significantly <span class="citation" data-cites="Foody2002 Radoux2014"><a href="#ref-Foody2002" role="doc-biblioref">[33]</a>, <a href="#ref-Radoux2014" role="doc-biblioref">[34]</a></span>. Validation at Level 2 also provides a solid foundation for publishing the final product at Level 1, since the latter represents a direct aggregation of already validated classes with greater specificity. This increases confidence in the results and provides a rigorous benchmark for future model improvements or additional applications in territorial, agricultural, or environmental analysis contexts.</p>
<div class="cell">
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Load the raster classified</span></span>
<span><span class="va">mz4</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rspatial.github.io/terra/reference/rast.html">rast</a></span><span class="op">(</span><span class="st">"data/ct_chile/Extras/SENTINEL-2_MSI_MOSAIC_2020-05-03_2021-05-22_class_v1.tif"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Level 2 reclassification matrix</span></span>
<span><span class="va">reclass_lv2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/matrix.html">matrix</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>  <span class="fl">1</span>, <span class="fl">81</span>,   <span class="fl">2</span>, <span class="fl">81</span>,   <span class="fl">3</span>, <span class="fl">81</span>,   <span class="fl">4</span>, <span class="fl">24</span>,</span>
<span>  <span class="fl">5</span>, <span class="fl">61</span>,   <span class="fl">6</span>, <span class="fl">61</span>,   <span class="fl">7</span>, <span class="fl">41</span>,   <span class="fl">8</span>, <span class="fl">21</span>,</span>
<span>  <span class="fl">9</span>, <span class="fl">21</span>,   <span class="fl">10</span>,<span class="fl">21</span>,  <span class="fl">11</span>,<span class="fl">22</span>,   <span class="fl">12</span>, <span class="fl">22</span>,</span>
<span>  <span class="fl">13</span>, <span class="fl">51</span>,  <span class="fl">14</span>,<span class="fl">72</span>,  <span class="fl">15</span>,<span class="fl">82</span>,   <span class="fl">16</span>, <span class="fl">23</span>,</span>
<span>  <span class="fl">17</span>, <span class="fl">23</span>,  <span class="fl">18</span>,<span class="fl">31</span>,  <span class="fl">19</span>,<span class="fl">31</span>,   <span class="fl">20</span>, <span class="fl">31</span>,</span>
<span>  <span class="fl">21</span>, <span class="fl">71</span>,  <span class="fl">22</span>,<span class="fl">71</span>,  <span class="fl">23</span>,<span class="fl">81</span>,   <span class="fl">24</span>, <span class="fl">11</span><span class="op">)</span>, ncol <span class="op">=</span> <span class="fl">2</span>, byrow <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Reclassify</span></span>
<span><span class="va">mz4_reclass_nv2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rspatial.github.io/terra/reference/classify.html">classify</a></span><span class="op">(</span><span class="va">mz4</span>, rcl <span class="op">=</span> <span class="va">reclass_lv2</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Export </span></span>
<span><span class="va">dir_out_rec</span>  <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_19HBA_out</span>, <span class="st">"/Reclass/"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html">dir.create</a></span><span class="op">(</span><span class="va">dir_out_rec</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span>, showWarnings <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/writeRaster.html">writeRaster</a></span><span class="op">(</span><span class="va">mz4_reclass_nv2</span>, filename <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_out_rec</span>, <span class="st">"SENTINEL-2_MSI_MOSAIC_2020-05-03_2021-05-22_class_v1.tif"</span><span class="op">)</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="sampling-design" class="level3" data-number="18.10.1"><h3 data-number="18.10.1" class="anchored" data-anchor-id="sampling-design">
<span class="header-section-number">18.10.1</span> Sampling design</h3>
<p>The first step in the sampling design is to load the reclassified MZ4 mosaic to Level 2. To do this, the cube label vector is defined, and the information is loaded from local files. To achieve proper visualization, the previously developed color palette is modified.</p>
<div class="cell">
<div class="sourceCode" id="cb27"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">cl_tbl_eng</span> <span class="op">&lt;-</span> <span class="fu">tibble</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/tibble.html">tibble</a></span><span class="op">(</span>name <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/character.html">character</a></span><span class="op">(</span><span class="op">)</span>, color <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/character.html">character</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">cl_tbl_eng</span> <span class="op">&lt;-</span> <span class="va">cl_tbl_eng</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Artificial_surfaces"</span>, color <span class="op">=</span> <span class="st">"#ff0000"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Temporary_crops"</span>,     color <span class="op">=</span> <span class="st">"#fbff01"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Permanent_crops"</span>,     color <span class="op">=</span> <span class="st">"#fb9a99"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Improved_pastures"</span>,   color <span class="op">=</span> <span class="st">"#8c20c9"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Fallow"</span>,              color <span class="op">=</span> <span class="st">"#7d4a0c"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Forest_plantation"</span>,   color <span class="op">=</span> <span class="st">"#95ffa8"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Water_bodies"</span>,        color <span class="op">=</span> <span class="st">"#204ee3"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Wetlands"</span>,            color <span class="op">=</span> <span class="st">"#93dfe6"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Native_forest"</span>,       color <span class="op">=</span> <span class="st">"#054d05"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Grasslands"</span>,          color <span class="op">=</span> <span class="st">"#01665e"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Shrublands"</span>,          color <span class="op">=</span> <span class="st">"#8b864e"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Non_vegetated_area"</span>,  color <span class="op">=</span> <span class="st">"#cccccc"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"Snow_glaciers"</span>,       color <span class="op">=</span> <span class="st">"#00bfff"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"No_Class"</span>,            color <span class="op">=</span> <span class="st">"white"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"No_Data"</span>,             color <span class="op">=</span> <span class="st">"white"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://tibble.tidyverse.org/reference/add_row.html">add_row</a></span><span class="op">(</span>name <span class="op">=</span> <span class="st">"No_Samples"</span>,          color <span class="op">=</span> <span class="st">"white"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Load the color table into `sits`</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_colors_set.html">sits_colors_set</a></span><span class="op">(</span><span class="va">cl_tbl_eng</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Vector of labels</span></span>
<span><span class="va">labels_lv2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>  <span class="st">"11"</span>  <span class="op">=</span> <span class="st">"Artificial_surfaces"</span>,          </span>
<span>  <span class="st">"21"</span>  <span class="op">=</span> <span class="st">"Temporary_crops"</span>,           </span>
<span>  <span class="st">"22"</span>  <span class="op">=</span> <span class="st">"Permanent_crops"</span>,           </span>
<span>  <span class="st">"23"</span>  <span class="op">=</span> <span class="st">"Improved_pastures"</span>,                   </span>
<span>  <span class="st">"24"</span>  <span class="op">=</span> <span class="st">"Fallow"</span>,             </span>
<span>  <span class="st">"31"</span>  <span class="op">=</span> <span class="st">"Forest_plantation"</span>,              </span>
<span>  <span class="st">"41"</span>  <span class="op">=</span> <span class="st">"Water_bodies"</span>,        </span>
<span>  <span class="st">"51"</span>  <span class="op">=</span> <span class="st">"Wetlands"</span>,        </span>
<span>  <span class="st">"61"</span>  <span class="op">=</span> <span class="st">"Native_forest"</span>,        </span>
<span>  <span class="st">"71"</span>  <span class="op">=</span> <span class="st">"Grasslands"</span>,       </span>
<span>  <span class="st">"72"</span>  <span class="op">=</span> <span class="st">"Shrublands"</span>,       </span>
<span>  <span class="st">"81"</span>  <span class="op">=</span> <span class="st">"Non_vegetated_area"</span>,                  </span>
<span>  <span class="st">"82"</span>  <span class="op">=</span> <span class="st">"Snow_glaciers"</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Load data cube from local files</span></span>
<span><span class="va">mz4_lv2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_cube.html">sits_cube</a></span><span class="op">(</span></span>
<span>  source     <span class="op">=</span> <span class="st">"MPC"</span>,</span>
<span>  collection <span class="op">=</span> <span class="st">"SENTINEL-2-L2A"</span>,</span>
<span>  data_dir   <span class="op">=</span> <span class="va">dir_out_rec</span>,</span>
<span>  bands      <span class="op">=</span> <span class="st">"class"</span>,</span>
<span>  labels     <span class="op">=</span> <span class="va">labels_lv2</span>,</span>
<span>  parse_info <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"satellite"</span>, <span class="st">"sensor"</span>, <span class="st">"tile"</span>,</span>
<span>    <span class="st">"start_date"</span>, <span class="st">"end_date"</span>, <span class="st">"band"</span>, <span class="st">"version"</span></span>
<span>  <span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Plot the classified mosaic</span></span>
<span><span class="fu"><a href="https://rspatial.github.io/terra/reference/plot.html">plot</a></span><span class="op">(</span><span class="va">mz4_lv2</span>, legend_position <span class="op">=</span> <span class="st">"outside"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-map_mz4" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/classified_mosaic_mz4.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.22: Classified mosaic for land cover in MZ4, for May 2020 to May 2021.</figcaption></figure>
</div>
</div>
</div>
<p>This method involves dividing the population into homogeneous subgroups (using Cochran’s methos for stratified random sampling), or strata, and then applying random sampling within each stratum. In the context of LULC mapping, the classification map serves as the basis for stratification, where each land cover class is treated as a distinct and homogeneous stratum.</p>
<p>For this purpose, we use the <code><a href="https://rdrr.io/pkg/sits/man/sits_sampling_design.html">sits_sampling_design()</a></code> function, which requires the prior definition of the <code>expected_ua</code> (user’s accuracy) for each class. In addition, it takes as parameters a <code>std_err</code> (standard error) of 0.01 and a <code>rare_class_prop</code> (proportion of rare classes) of 0.1, respectively. Three sample allocation scenarios, 75, 100, and 120 samples per class, are to be generated for subsequent evaluation and selection.</p>
<div class="cell">
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Vector of labels</span></span>
<span><span class="va">labels_lv2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span>                                </span>
<span>  <span class="st">"11"</span>  <span class="op">=</span> <span class="st">"Artificial_surfaces"</span>,          </span>
<span>  <span class="st">"21"</span>  <span class="op">=</span> <span class="st">"Temporary_crops"</span>,           </span>
<span>  <span class="st">"22"</span>  <span class="op">=</span> <span class="st">"Permanent_crops"</span>,           </span>
<span>  <span class="st">"23"</span>  <span class="op">=</span> <span class="st">"Improved_pastures"</span>,                   </span>
<span>  <span class="st">"24"</span>  <span class="op">=</span> <span class="st">"Fallow"</span>,             </span>
<span>  <span class="st">"31"</span>  <span class="op">=</span> <span class="st">"Forest_plantation"</span>,              </span>
<span>  <span class="st">"41"</span>  <span class="op">=</span> <span class="st">"Water_bodies"</span>,        </span>
<span>  <span class="st">"51"</span>  <span class="op">=</span> <span class="st">"Wetlands"</span>,        </span>
<span>  <span class="st">"61"</span>  <span class="op">=</span> <span class="st">"Native_forest"</span>,        </span>
<span>  <span class="st">"71"</span>  <span class="op">=</span> <span class="st">"Grasslands"</span>,       </span>
<span>  <span class="st">"72"</span>  <span class="op">=</span> <span class="st">"Shrublands"</span>,       </span>
<span>  <span class="st">"81"</span>  <span class="op">=</span> <span class="st">"Non_vegetated_area"</span>,                  </span>
<span>  <span class="st">"82"</span>  <span class="op">=</span> <span class="st">"Snow_glaciers"</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Load data cube from local files</span></span>
<span><span class="va">mz4_lv2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_cube.html">sits_cube</a></span><span class="op">(</span></span>
<span>  source     <span class="op">=</span> <span class="st">"MPC"</span>,</span>
<span>  collection <span class="op">=</span> <span class="st">"SENTINEL-2-L2A"</span>,</span>
<span>  data_dir   <span class="op">=</span> <span class="va">dir_out_rec</span>,</span>
<span>  bands      <span class="op">=</span> <span class="st">"class"</span>,</span>
<span>  labels     <span class="op">=</span> <span class="va">labels_lv2</span>,</span>
<span>  parse_info <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"satellite"</span>, <span class="st">"sensor"</span>, <span class="st">"tile"</span>,</span>
<span>    <span class="st">"start_date"</span>, <span class="st">"end_date"</span>, <span class="st">"band"</span>, <span class="st">"version"</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Sampling design (Level 2)</span></span>
<span><span class="va">sampling_design</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_sampling_design.html">sits_sampling_design</a></span><span class="op">(</span></span>
<span>    cube            <span class="op">=</span> <span class="va">mz4_lv2</span>,</span>
<span>    expected_ua     <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>        Artificial_surfaces <span class="op">=</span> <span class="fl">0.75</span>,</span>
<span>        Temporary_crops     <span class="op">=</span> <span class="fl">0.8</span>,        </span>
<span>        Permanent_crops     <span class="op">=</span> <span class="fl">0.8</span>,</span>
<span>        Improved_pastures   <span class="op">=</span> <span class="fl">0.8</span>,</span>
<span>        Fallow              <span class="op">=</span> <span class="fl">0.8</span>, </span>
<span>        Forest_plantation   <span class="op">=</span> <span class="fl">0.75</span>,</span>
<span>        Water_bodies        <span class="op">=</span> <span class="fl">0.75</span>, </span>
<span>        Wetlands            <span class="op">=</span> <span class="fl">0.7</span>,</span>
<span>        Native_forest       <span class="op">=</span> <span class="fl">0.75</span>,</span>
<span>        Grasslands          <span class="op">=</span> <span class="fl">0.7</span>,</span>
<span>        Shrublands          <span class="op">=</span> <span class="fl">0.7</span>,</span>
<span>        Non_vegetated_area  <span class="op">=</span> <span class="fl">0.75</span>,</span>
<span>        Snow_glaciers       <span class="op">=</span> <span class="fl">0.7</span></span>
<span>    <span class="op">)</span>,</span>
<span>    std_err         <span class="op">=</span> <span class="fl">0.01</span>,</span>
<span>    alloc_options   <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="fl">120</span>, <span class="fl">100</span>, <span class="fl">75</span><span class="op">)</span>,</span>
<span>    rare_class_prop <span class="op">=</span> <span class="fl">0.1</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Show sampling design</span></span>
<span><span class="va">sampling_design</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>                    prop         expected_ua std_dev equal alloc_120 alloc_100
Artificial_surfaces 0.01102111   0.75        0.433   143   120       100      
Temporary_crops     0.06698658   0.8         0.4     143   120       100      
Permanent_crops     0.05512354   0.8         0.4     143   120       100      
Improved_pastures   0.1025799    0.8         0.4     143   132       172      
Fallow              0.007910668  0.8         0.4     143   120       100      
Forest_plantation   0.2326992    0.75        0.433   143   300       390      
Water_bodies        0.007476513  0.75        0.433   143   120       100      
Wetlands            0.0006144286 0.7         0.458   143   120       100      
Native_forest       0.1782748    0.75        0.433   143   229       299      
Grasslands          0.09582032   0.7         0.458   143   120       100      
Shrublands          0.09356777   0.7         0.458   143   120       100      
Non_vegetated_area  0.09586879   0.75        0.433   143   120       100      
Snow_glaciers       0.05205643   0.7         0.458   143   120       100      
                    alloc_75 alloc_prop
Artificial_surfaces 75       21        
Temporary_crops     75       125       
Permanent_crops     75       103       
Improved_pastures   222      191       
Fallow              75       15        
Forest_plantation   503      433       
Water_bodies        75       14        
Wetlands            75       1         
Native_forest       386      332       
Grasslands          75       178       
Shrublands          75       174       
Non_vegetated_area  75       178       
Snow_glaciers       75       97        </code></pre>
</div>
</div>
</section><section id="sec-stratified-random-sampling" class="level3" data-number="18.10.2"><h3 data-number="18.10.2" class="anchored" data-anchor-id="sec-stratified-random-sampling">
<span class="header-section-number">18.10.2</span> Stratified random sampling</h3>
<p>The next step is to select a sampling design option to generate a set of points through stratified sampling. These points will later serve as the basis for accuracy assessment. In this case, we select the <code>alloc_100</code> option from the sampling design to generate a set of stratified samples and save the results in a shp file.</p>
<div class="cell">
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># select samples</span></span>
<span><span class="va">samples</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_stratified_sampling.html">sits_stratified_sampling</a></span><span class="op">(</span>cube <span class="op">=</span> <span class="va">mz4_lv2</span>,</span>
<span>                                    <span class="va">sampling_design</span>, <span class="st">"alloc_100"</span>,</span>
<span>                                    shp_file <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="va">dir_19HBA_out</span>, <span class="st">"/samples_MZ4_nv2.shp"</span><span class="op">)</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>The resulting output is a shapefile containing point features with geographic coordinates (latitude and longitude) and their corresponding class labels derived from the classified map. During sample generation, an oversampling parameter is applied to address uncertainties in areas such as class boundaries or areas with high classification ambiguity. This strategy enables the exclusion of points with uncertain class assignments, thereby enhancing the reliability of the validation process. By avoiding ambiguous samples, the final set of points more accurately represents the reference classes and reduces potential bias introduced by misclassified pixels.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-sample_design" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/map_sample_design.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.23: Spatial distribution of the generated sampling design (Source: National Institute of Statistics, Chile).</figcaption></figure>
</div>
</div>
</div>
<p>After reviewing and labeling the sampling points in QGIS, a shp object was generated containing the ground-truth label for each point corresponding to the study season.</p>
</section><section id="accuracy-assessment-of-classified-sampling" class="level3" data-number="18.10.3"><h3 data-number="18.10.3" class="anchored" data-anchor-id="accuracy-assessment-of-classified-sampling">
<span class="header-section-number">18.10.3</span> Accuracy assessment of classified sampling</h3>
<p>To measure the accuracy of classified images, an area weighting technique is used. The need for area-weighted estimates arises because the classes of uses and/or coverages are not evenly distributed across space.</p>
<p>To produce adjusted area estimates for classified maps, the <code><a href="https://rdrr.io/pkg/sits/man/sits_accuracy.html">sits_accuracy()</a></code> function is used. This function requires two main inputs: the classified data cube and a set of well-selected reference points obtained through stratified random sampling.</p>
<div class="cell">
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">mz4_val_lv2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r-spatial.github.io/sf/reference/st_read.html">st_read</a></span><span class="op">(</span><span class="st">"data/ct_chile/Samples/samples_validation_mz4.shp"</span><span class="op">)</span> </span>
<span></span>
<span><span class="va">area_acc</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/sits/man/sits_accuracy.html">sits_accuracy</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">mz4_lv2</span>, </span>
<span>  validation <span class="op">=</span> <span class="va">mz4_val_lv2</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Print the area estimated accuracy</span></span>
<span><span class="va">area_acc</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Area Weighted Statistics
Overall Accuracy = 0.92

Area-Weighted Users and Producers Accuracy
                    User Producer
Artificial_surfaces 0.84     1.00
Temporary_crops     1.00     0.83
Permanent_crops     0.71     0.86
Improved_pastures   0.78     0.95
Fallow              0.82     0.98
Forest_plantation   0.95     0.90
Water_bodies        0.94     1.00
Wetlands            0.78     1.00
Native_forest       0.97     0.96
Grasslands          0.99     0.95
Shrublands          0.82     0.87
Non_vegetated_area  1.00     0.92
Snow_glaciers       0.86     1.00

Mapped Area x Estimated Area (ha)
                    Mapped Area (ha) Error-Adjusted Area (ha)
Artificial_surfaces         47838.87                 40081.21
Temporary_crops            290765.80                350915.74
Permanent_crops            239272.40                199545.73
Improved_pastures          445264.02                363767.79
Fallow                      34337.50                 28671.17
Forest_plantation         1010067.34               1071241.09
Water_bodies                32452.98                 30543.98
Wetlands                     2667.02                  2074.35
Native_forest              773829.84                785940.71
Grasslands                 415923.14                433571.56
Shrublands                 406145.63                384067.89
Non_vegetated_area         416133.55                454642.67
Snow_glaciers              225959.11                195593.30
                    Conf Interval (ha)
Artificial_surfaces            3295.30
Temporary_crops               21397.29
Permanent_crops               26315.61
Improved_pastures             29258.14
Fallow                         2637.04
Forest_plantation             39188.45
Water_bodies                   1377.78
Wetlands                        527.08
Native_forest                 21684.53
Grasslands                    16649.55
Shrublands                    34881.45
Non_vegetated_area            14373.72
Snow_glaciers                 14027.70</code></pre>
</div>
</div>
<p>The results show an overall accuracy of 92%, demonstrating good performance of the classification model based on satellite image time series. Stable classes, such as native forest, water bodies, and artificial surfaces, exhibit high user and producer accuracy values, reflecting a clear spectro-temporal distinction throughout the year. On the other hand, agricultural classes, while also achieving good levels of accuracy, exhibit certain differences between the accuracy metrics, especially in permanent crops and improved pastures. This may be associated with intra-class variability or the coexistence of land covers with similar temporal signatures.</p>
<p>The comparison between the pixel-mapped area and the error-adjusted area reveals differences for several classes (<a href="#fig-area_est">Figure&nbsp;<span>18.24</span></a>), particularly those with lower individual accuracy. While a strictly linear relationship between class accuracy and the required adjustment level is not observed, trends are identified: classes with major discrepancies between user and producer accuracy (such as permanent crops and improved pastures) show more substantial adjustments, reflecting the presence of systematic errors of omission and commission. In contrast, stable and well-defined spectro-temporal classes, such as native forest, forest plantations, or water bodies, show smaller differences between the two estimates. Additionally, the figure includes error bars representing confidence intervals, and it is worth noting that the coefficient of variation (CV) can also be computed. Both the confidence interval and the CV are quality indicators widely used by national statistical offices (NSOs) when releasing survey-based estimates, as they provide transparent metrics of reliability. What is novel in this context is that such measures are not commonly applied in the field of Land Cover and Land Use (LCLU) mapping, despite their relevance for ensuring statistical rigor.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-area_est" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="./images/ct_chile/area_nv2_mz4.png" class="img-fluid figure-img" style="width:100.0%"></p>
<figcaption class="figure-caption">Figure&nbsp;18.24: Estimated area (in hectares) for each Level 2 land cover class in Macrozone 4. Light bars represent the area derived from pixel counts, while dark bars indicate the area adjusted using accuracy assessment results. Error bars correspond to the confidence intervals of the adjusted estimates.</figcaption></figure>
</div>
</div>
</div>
</section></section><section id="discussion" class="level2" data-number="18.11"><h2 data-number="18.11" class="anchored" data-anchor-id="discussion">
<span class="header-section-number">18.11</span> Discussion</h2>
<section id="institutional-relevance-and-purpose-of-the-map" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="institutional-relevance-and-purpose-of-the-map">Institutional relevance and purpose of the map</h3>
<p>The development of the new Land Use and Land Cover (LULC) map with an agricultural focus addresses a critical gap in Chile’s agricultural statistical system: the lack of a cartographic product explicitly designed to support official agricultural statistics with national coverage and high spatial precision. Unlike previous initiatives, such as <span class="citation" data-cites="Zhao2016"><a href="#ref-Zhao2016" role="doc-biblioref">[35]</a></span>, CLDynamicLandCover <span class="citation" data-cites="Galleguillos2024"><a href="#ref-Galleguillos2024" role="doc-biblioref">[36]</a></span>, <a href="https://chile.mapbiomas.org/descripcion-general-de-la-metodologia/">MapBiomas Chile</a>, or global products like <a href="https://esa-worldcover.org/en">WorldCover</a> and <a href="https://www.copernicuslac-chile.eu/uso-y-cobertura-de-suelos/">Copernicus LAC</a>, this map originates from within the national statistical system itself. Its main objectives are to strengthen the agricultural sampling frame, improve the estimation of cultivated areas, and enhance operational efficiency between censuses. This initiative positions the National Institute of Statistics (INE) as a key agent in the modernization of agricultural statistics, in line with international recommendations to integrate Earth observation into official data systems <span class="citation" data-cites="FAO2017 tasktea"><a href="#ref-FAO2017" role="doc-biblioref">[37]</a>, <a href="#ref-tasktea" role="doc-biblioref">[38]</a></span>.</p>
<p>From a methodological perspective, the INE LULC map incorporates time series of satellite imagery, representing a significant advancement over traditional approaches based on single-date imagery or seasonal composites <span class="citation" data-cites="Zhang2024"><a href="#ref-Zhang2024" role="doc-biblioref">[39]</a></span>. This strategy enables the capture of intra-annual crop phenology, including practices such as successive plantings, fallow periods, and seasonal cycles. Combined with a spatial resolution of 10 meters, the approach enables the detection of small agricultural plots and within-field heterogeneity, thereby enhancing the representativeness and accuracy of cultivated area estimates <span class="citation" data-cites="Ren2022"><a href="#ref-Ren2022" role="doc-biblioref">[40]</a></span>. In contrast to global products with generic legends, the INE classification scheme is tailored to Chile’s productive landscape and institutional standards, ensuring that the generated information is both technically robust and operationally relevant for the design, stratification, and updating of agricultural surveys. Furthermore, the approach is structured around macrozones (a zonation of the country), which facilitate optimal scalability of the process at the national level.</p>
</section><section id="classification-and-sampling-scheme-in-lulc-mapping" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="classification-and-sampling-scheme-in-lulc-mapping">Classification and sampling scheme in LULC mapping</h3>
<p>The LULC classification scheme defined for this work follows a three-level hierarchical structure, primarily based on the phenological and temporal characteristics of agricultural classes. This methodological decision stems from lessons learned during the pilot phase conducted in the Maule Region, where the classification relied solely on agronomic criteria. Unlike that approach, the current scheme acknowledges the dynamic nature of certain land cover types, replacing the concept of static cover with one that incorporates interannual variability. As such, it includes classes such as winter and spring crops, as well as deciduous and evergreen perennial species, according to their phenological cycles, which enhances the representativeness of agricultural classes <span class="citation" data-cites="Zhou2024 Wasniewski2022"><a href="#ref-Zhou2024" role="doc-biblioref">[18]</a>, <a href="#ref-Wasniewski2022" role="doc-biblioref">[41]</a></span>.</p>
<p>The hierarchical design enables multiple applications throughout the workflow. The 24 Level 3 classes were used to build training samples and train the classification model. The 13 Level 2 classes were employed for statistical validation of the map, following best practices to ensure enough samples per category. Finally, the eight Level 1 classes were designed to align with the official CCUS 1.0 classification scheme of the INE, ensuring institutional compatibility. This hierarchical approach is widely recommended in the LULC classification literature for its ability to improve thematic accuracy, reduce class confusion, and facilitate multi-scale comparisons <span class="citation" data-cites="Wasniewski2022"><a href="#ref-Wasniewski2022" role="doc-biblioref">[41]</a></span>.</p>
<p>The use of satellite image time series was essential for constructing both the classification scheme and the training samples, as it enabled the identification of phenological differences among agricultural classes. This temporal dimension is particularly useful for distinguishing spectrally similar land cover types with different growth cycles, such as winter vs.&nbsp;spring crops, or deciduous vs.&nbsp;evergreen fruit trees. Phenology serves as a distinctive signature that enhances spectral-temporal separability, as broadly reported in the literature<span class="citation" data-cites="Wardlow2007 Wardlow2008"><a href="#ref-Wardlow2008" role="doc-biblioref">[16]</a>, <a href="#ref-Wardlow2007" role="doc-biblioref">[42]</a></span>.</p>
<p>To construct the training samples, representative polygons for each Level 3 class were delineated using cadastral data, administrative records, and NDVI time-series analysis. From these polygons, labeled points were extracted using, for the first time, a sampling strategy proportional to polygon area rather than a fixed number of points. Minimum and maximum point thresholds were set, and polygons larger than 25 hectares were excluded. This approach yielded a more representative training dataset in terms of the spatial distribution of land cover types <span class="citation" data-cites="Zhang2023"><a href="#ref-Zhang2023" role="doc-biblioref">[43]</a></span>.</p>
</section><section id="construction-of-macrozones-as-a-basis-for-territorial-modeling" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="construction-of-macrozones-as-a-basis-for-territorial-modeling">Construction of macrozones as a basis for territorial modeling</h3>
<p>Dividing the national territory into homogeneous zones is a key strategy for improving both the accuracy and efficiency of LULC mapping. Stratifying the study area into ecologically similar regions helps reduce spectral heterogeneity, allowing classifiers to better adapt to local environmental conditions, such as climate, vegetation, or topography, which in turn enhances classification accuracy <span class="citation" data-cites="Kavhu2021 Ojwang2024"><a href="#ref-Ojwang2024" role="doc-biblioref">[11]</a>, <a href="#ref-Kavhu2021" role="doc-biblioref">[44]</a></span>. In geographically diverse contexts like Chile, this type of zonation is particularly advantageous, as it enables the analysis of spectrally coherent imagery and improves the representativeness of sampling designs for calibration and validation, ensuring a more equitable and efficient distribution of samples across both geographic and spectral spaces <span class="citation" data-cites="Dong2022"><a href="#ref-Dong2022" role="doc-biblioref">[45]</a></span>.</p>
<p>Additionally, subdividing the territory supports the detection of localized patterns and dynamics, such as deforestation, urban expansion, or crop rotation, that may be obscured in broader-scale analyses. From a technical standpoint, territorial fragmentation also optimizes the use of computational resources by reducing task complexity, enabling parallel processing, and significantly shortening computing times <span class="citation" data-cites="Ojwang2024 Loveland2004 Hansen2013"><a href="#ref-Ojwang2024" role="doc-biblioref">[11]</a>, <a href="#ref-Loveland2004" role="doc-biblioref">[46]</a>, <a href="#ref-Hansen2013" role="doc-biblioref">[47]</a></span>. In practice, this approach allows the independent execution of the LULC classification workflow within each territorial unit, defined at a scale smaller tgan national but larger than regional division, thus ensuring that satellite data inputs can be processed effectively without exceeding the capabilities or processing time constraints of a dedicated computing</p>
</section><section id="inclusion-of-the-dem-as-a-feature-for-modeling" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="inclusion-of-the-dem-as-a-feature-for-modeling">Inclusion of the DEM as a feature for modeling</h3>
<p>An important finding from the modeling process, as well as from the preceding pilot study, was the contribution of including topographic variables derived from Digital Elevation Models (DEMs) in the classification. In a mountainous country like Chile, elevation and slope strongly influence vegetation distribution and agricultural practices, introducing spectral variations associated with climate and terrain <span class="citation" data-cites="Luebert2004"><a href="#ref-Luebert2004" role="doc-biblioref">[48]</a></span>. Integrating these factors into the model significantly improved classification accuracy, in line with findings reported by several authors<span class="citation" data-cites="Bilucan2021 PachecoPrado2024 Pizarro2022"><a href="#ref-Bilucan2021" role="doc-biblioref">[49]</a>–<a href="#ref-Pizarro2022" role="doc-biblioref">[51]</a></span>.</p>
<p>Our experience confirms that Chile’s mountainous geography benefits from this approach, as expected based on international evidence. National initiatives, such as those by <span class="citation" data-cites="Zhao2016"><a href="#ref-Zhao2016" role="doc-biblioref">[35]</a></span>, <span class="citation" data-cites="Galleguillos2024"><a href="#ref-Galleguillos2024" role="doc-biblioref">[36]</a></span>, and <a href="https://chile.mapbiomas.org/descripcion-general-de-la-metodologia/">MapBiomas Chile</a>, have already incorporated elevation data into their methodologies, recognizing that the inclusion of altitudinal gradients susbtantially enhances the discrimination of classes influenced by topography, for example, distinguishing between valley vegetation and high-altitude vegetation.</p>
</section><section id="analysis-of-the-quality-of-training-data" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="analysis-of-the-quality-of-training-data">Analysis of the quality of training data</h3>
<p>One of the main methodological challenges addressed was the handling of potentially noisy or atypical training data. Using Self-Organizing Maps (SOM), it was possible to identify internal dissimilarities within a subset of samples <span class="citation" data-cites="Santos2021"><a href="#ref-Santos2021" role="doc-biblioref">[28]</a></span>, indicating the presence of noise. Although the sits package allows for the automatic filtering of such observations, we opted to retain them to avoid exacerbating class imbalance, which could have resulted in the complete loss of a minority class. Recent studies warn that severely imbalanced training sets can bias model learning <span class="citation" data-cites="Johnson2019 Mujahid2024"><a href="#ref-Johnson2019" role="doc-biblioref">[32]</a>, <a href="#ref-Mujahid2024" role="doc-biblioref">[52]</a></span>. For this reason, we adopted an alternative approach inspired by <span class="citation" data-cites="deSimone2025"><a href="#ref-deSimone2025" role="doc-biblioref">[30]</a></span>, replacing filtering with a combined statistical balancing strategy: oversampling of minority classes using SMOTE and undersampling of dominant classes using SOM, a technique that reduces sampling variance and bias while preserving the intrinsic statistical structure of the original dataset <span class="citation" data-cites="May2010 RuizMoreno2023"><a href="#ref-May2010" role="doc-biblioref">[53]</a>, <a href="#ref-RuizMoreno2023" role="doc-biblioref">[54]</a></span>.</p>
<p>This strategy enabled the retention of all classes, generating synthetic examples where necessary without discarding valuable information. Upon re-evaluating internal consistency through cluster analysis, several cover types showed notable improvements: snow, wetlands, and water bodies achieved cluster purities between 97% and 100%, while complex agricultural and forest classes, such as successive crops, native forest, or harvested plantations, exceeded 83% purity. These results suggest that balancing contributed to a more consolidated spectral-temporal representation of these categories. However, some dominant classes experienced slight decreases in purity, likely due to the increased diversity introduced by oversampling or even the potential injection of synthetic noise <span class="citation" data-cites="Douzas2019"><a href="#ref-Douzas2019" role="doc-biblioref">[55]</a></span>. This was the case for perennial fruit crops, overlapping with closely related classes such as forest plantations and deciduous forests. These residual confusions highlight the persistent difficulty of separating phenologically or structurally similar classes while also offering clear insight into which categories could benefit from additional discriminative variables, such as SAR bands or textural metrics <span class="citation" data-cites="Xu2024"><a href="#ref-Xu2024" role="doc-biblioref">[56]</a></span>. In the future, we propose evaluating more sophisticated techniques such as Geometric SMOTE, which could allow for the generation of more coherent synthetic samples and further improvement in training consistency <span class="citation" data-cites="Douzas2019 Douzas2019a"><a href="#ref-Douzas2019" role="doc-biblioref">[55]</a>, <a href="#ref-Douzas2019a" role="doc-biblioref">[57]</a></span>.</p>
</section><section id="model-evaluation" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="model-evaluation">Model evaluation</h3>
<p>The classification model achieved a globally satisfactory performance, with an area-weighted overall accuracy of 0.92, validating both the use of satellite image time series and the improvements applied in the preparation and balancing of the training dataset <span class="citation" data-cites="Simoes2021 Santos2021"><a href="#ref-Simoes2021" role="doc-biblioref">[24]</a>, <a href="#ref-Santos2021" role="doc-biblioref">[28]</a></span>. However, a disaggregated analysis by class reveals disparities: agricultural classes such as <em>Permanent crops</em> and <em>Improved Pastures</em> exhibit commission errors (low User’s Accuracy and high Producer’s Accuracy), while <em>Temporary crops</em> show omission errors (low Producer’s Accuracy and high User’s Accuracy), reflecting limitations in the spectral-temporal separability of certain land cover types. These inconsistencies carry over into area estimations, where differences were observed between pixel-count-based mapped areas and validation-adjusted areas, particularly for classes such as <em>Forest plantation</em>, <em>Temporary crops</em>, and <em>Shrublands</em>.</p>
<p>To correct these biases, an adjusted area estimation was implemented following the statistical framework proposed by <span class="citation" data-cites="Olofsson2014"><a href="#ref-Olofsson2014" role="doc-biblioref">[27]</a></span>, which accounts for errors derived from the confusion matrix and employs a stratified random sampling design. This approach produced more accurate area estimates, incorporating confidence intervals, and enhancing the transparency of the final product <span class="citation" data-cites="EuropeanCommission2021"><a href="#ref-EuropeanCommission2021" role="doc-biblioref">[58]</a></span>. Despite the overall improvement, challenges remain in differentiating spectrally or phenologically similar classes, such as <em>Permanent crops</em> and <em>Forest plantation</em>, highlighting the need to integrate additional discriminative variables, such as SAR bands or textural metrics, to improve class separability <span class="citation" data-cites="Xu2024"><a href="#ref-Xu2024" role="doc-biblioref">[56]</a></span>. In this way, the adopted approach not only ensures statistical rigor and traceability but also pinpoints priority areas for future model refinement.</p>
</section><section id="advanced-applications-of-remote-sensing-and-deep-learning-in-official-statistics" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="advanced-applications-of-remote-sensing-and-deep-learning-in-official-statistics">Advanced applications of remote sensing and deep learning in official statistics</h3>
<p>One of the most promising improvements to be incorporated in future stages of this work is the use of deep learning techniques, which offer the ability to process satellite time series data while accounting for the temporal sequence of observations. This capability allows for the explicit modeling of spectral dynamics throughout the year, particularly valuable for land cover classes with strong seasonality or well-defined phenological cycles <span class="citation" data-cites="Russwurm2020a Tsuchiya2025 Wang2023"><a href="#ref-Russwurm2020a" role="doc-biblioref">[59]</a>–<a href="#ref-Wang2023" role="doc-biblioref">[61]</a></span>. Numerous studies have demonstrated that deep neural networks such as 3D Convolutional Neural Networks (3D-CNNs) or Long Short-Term Memory (LSTM) models outperform traditional algorithms like Random Forests or Support Vector Machines in multitemporal classification tasks for crops and land cover <span class="citation" data-cites="Zhong2019 Pelletier2019"><a href="#ref-Zhong2019" role="doc-biblioref">[62]</a>, <a href="#ref-Pelletier2019" role="doc-biblioref">[63]</a></span>.</p>
<p>Beyond deep learning, remote sensing offers a broad set of tools with high potential for enhancing the production of official statistics. Below are key applications with relevance to agriculture.</p>
<p>First, remote sensing can be used to automatically map rural plots, which would streamline census and survey operations and facilitate the registration of farmers and production units <span class="citation" data-cites="Davidson2017"><a href="#ref-Davidson2017" role="doc-biblioref">[4]</a></span>.</p>
<p>Additionally, satellite-based crop type mapping enables the generation of highly accurate statistics on cultivated areas by capturing the phenological development of crops. These multitemporal classifications are crucial for designing more robust sampling frames for agricultural surveys, and they have proven effective in monitoring intra- and interannual shifts in cropping patterns <span class="citation" data-cites="Mcnairn2009 PenaBarragan2011"><a href="#ref-Mcnairn2009" role="doc-biblioref">[64]</a>, <a href="#ref-PenaBarragan2011" role="doc-biblioref">[65]</a></span>.</p>
<p>Remote sensing also makes it possible to estimate crop yields, using either statistical regression models or simulations based on spectral indices. These yield estimations can complement traditional surveys, especially in data-scarce regions, and have been applied by FAO and programs such as AMIS and GEOGLAM for food security monitoring <span class="citation" data-cites="FAO2017 Ambrosio2024"><a href="#ref-FAO2017" role="doc-biblioref">[37]</a>, <a href="#ref-Ambrosio2024" role="doc-biblioref">[66]</a></span>.</p>
<p>Moreover, remote sensing techniques can significantly contribute to other domains of official statistics, such as:</p>
<ul>
<li>
<strong>Sample design</strong>, through land use-based stratification and the delineation of primary sampling units <span class="citation" data-cites="Gallego2017"><a href="#ref-Gallego2017" role="doc-biblioref">[2]</a></span>.</li>
</ul>
<!-- --><ul>
<li>
<strong>Disaster mapping and forecasting</strong>, by using optical and radar sensors to identify areas affected by droughts, wildfires, or floods <span class="citation" data-cites="Rojas2017"><a href="#ref-Rojas2017" role="doc-biblioref">[67]</a></span>.</li>
</ul>
<!-- --><ul>
<li>
<strong>Environmental and climate statistics</strong>, including deforestation monitoring, soil moisture, albedo, and climate change indicators <span class="citation" data-cites="Achard2017"><a href="#ref-Achard2017" role="doc-biblioref">[68]</a></span>.</li>
</ul>
<!-- --><ul>
<li><p><strong>Urban statistics</strong>, using high-resolution imagery to map urban expansion, infrastructure, and population density <span class="citation" data-cites="Grippa2019"><a href="#ref-Grippa2019" role="doc-biblioref">[69]</a></span>.</p></li>
<li><p><strong>Operational efficiency</strong>, by reducing the cost and time of field data collection while improving data quality and coverage <span class="citation" data-cites="Delince2017"><a href="#ref-Delince2017" role="doc-biblioref">[3]</a></span>.</p></li>
</ul></section><section id="limitations-and-opportunities-for-improvement" class="level3 unnumbered"><h3 class="unnumbered anchored" data-anchor-id="limitations-and-opportunities-for-improvement">Limitations and opportunities for improvement</h3>
<p>Despite achieving satisfactory results in terms of evaluation metrics, this process presented certain methodological and technical limitations, as well as clear opportunities to enhance its precision and robustness in future versions.</p>
<p><strong>Limitations</strong></p>
<p>One of the main limitations relates to the spectral nature of the input data. Relying exclusively on optical bands and variables derived from the digital elevation model limited the model’s ability to discriminate between certain thematic classes, particularly in areas where land covers share similar spectral characteristics, such as perennial fruit trees and forest plantations.</p>
<p>Additionally, during the exploration of improvements for MZ4, the integration of radar imagery was considered as a complementary data source. However, radar images with radiometric correction were not available for that area, which prevented their inclusion in this version.</p>
<p>Furthermore, persistent cloud cover in some macrozones, particularly in southern Chile and coastal areas, can restrict the availability of usable optical satellite imagery, especially during the winter months, limiting the density and quality of the time series.</p>
<p><strong>Opportunities for Improvement</strong></p>
<p>Key opportunities for improvement include integrating new data sources into the satellite data cube, particularly radar-derived variables (e.g., from Sentinel-1), as well as incorporating texture metrics computed from optical imagery.</p>
<p>Radar data would contribute an additional structural dimension to the analysis, as it captures information on terrain roughness, vegetation structure, and soil moisture. Importantly, radar imagery is not affected by atmospheric conditions, making it highly valuable in cloud-prone areas where optical data availability is limited <span class="citation" data-cites="Tsuchiya2025 Xu2024"><a href="#ref-Xu2024" role="doc-biblioref">[56]</a>, <a href="#ref-Tsuchiya2025" role="doc-biblioref">[60]</a></span>.</p>
<p>Meanwhile, the inclusion of texture metrics allows for the quantification of spatial patterns within land covers, facilitating the differentiation of spectrally similar but spatially distinct classes. Texture features, commonly derived using methods such as the Gray Level Co-occurrence Matrix (GLCM), have been shown to improve the discrimination of agricultural, forested, and transitional land cover types <span class="citation" data-cites="Ozesmi2002 Liu2019"><a href="#ref-Ozesmi2002" role="doc-biblioref">[70]</a>, <a href="#ref-Liu2019" role="doc-biblioref">[71]</a></span>.</p>
<p>This project marks a significant step forward in the application of remote sensing for the classification of agricultural land cover and its integration into official statistics. However, its future success will depend on the ability to overcome the identified limitations and realize the outlined improvement opportunities.</p>
</section></section><section id="conclusions" class="level2" data-number="18.12"><h2 data-number="18.12" class="anchored" data-anchor-id="conclusions">
<span class="header-section-number">18.12</span> Conclusions</h2>
<p>The work carried out by the Chilean National Statistics Institute demonstrates key methods for using remote sensing to generate reliable, high-quality agricultural statistics. By adopting a hierarchical-phenological classification scheme, it accurately captures the dynamic nature of the agricultural soils in the territory, including crop rotations, fallow periods, and/or permanent crop areas that are not reflected in static classifications and are crucial for the proper design of sampling frames.</p>
<p>The generation of training samples is one of the critical aspects of the workflow for obtaining land use and land cover maps. To use photointerpretation, NDVI curve analysis, RGB seasonal compositions, and the creation of labeled points for machine learning model training, a technique was devised to leverage the information that was available, both inside and outside the organization. It is also highlighted that the combination of satellite image time series with topographic information significantly improves thematic accuracy, especially in geographically complex environments.</p>
<p>This work also confirms that quality control and training data balancing procedures contribute to more generalizable models, while analysis tools such as SOM clusters help diagnose where challenges persist in the training data. The creation of this LULC map with an agronomic focus represents a strategic advance for agricultural statistics in Chile. Its benefits are evident: it provides updated and comprehensive inputs for designing more efficient sampling frames and surveys, as evidenced by the reviewed literature on high-temporal-resolution thematic maps.</p>
<p>The results reaffirm the importance of integrating new remote sensing methodologies in the production of agricultural statistics and offer new insights, from the value of phenological curves for discriminating crops to the importance of topographic data in complex landscapes. Looking ahead, scaling up this approach nationwide will strengthen the precision and reliability of agricultural area delimitation. This requires expanding predictor variables and, when feasible, exploring deep learning techniques to further enhance future versions.</p>
</section><section id="references" class="level2 unnumbered"><h2 class="unnumbered anchored" data-anchor-id="references">References</h2>


<div id="refs" class="references csl-bib-body" role="list">
<div id="ref-INE2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">[1] </div>
<div class="csl-right-inline">I. N. de E. INE, <span>“Documento de trabajo. Elaboración de un mapa de uso de suelo a partir de imágenes satelitales: Caso de estudio en la Región del Maule,”</span> Santiago, Chile, May 2025. [Online]. Available: <a href="https://www.ine.gob.cl/docs/default-source/documentos-de-trabajo/20250527_documento_trabajo_piloto_lc.pdf?sfvrsn=8d17acd9_2">https://www.ine.gob.cl/docs/default-source/documentos-de-trabajo/20250527_documento_trabajo_piloto_lc.pdf?sfvrsn=8d17acd9_2</a>.</div>
</div>
<div id="ref-Gallego2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[2] </div>
<div class="csl-right-inline">J. Gallego, <span>“Use of remote sensing for the design of sampling frames,”</span> J. Delincé, Ed. Rome: Handbook of the Global Strategy to improve Agricultural; Rural Statistics (GSARS), 2017.</div>
</div>
<div id="ref-Delince2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[3] </div>
<div class="csl-right-inline">J. Delincé, <span>“The cost-effectiveness of remote sensing in agricultural statistics,”</span> J. Delincé, Ed. Rome: Handbook of the Global Strategy to improve Agricultural; Rural Statistics (GSARS), 2017.</div>
</div>
<div id="ref-Davidson2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[4] </div>
<div class="csl-right-inline">A. Davidson, T. Fisette, H. McNairn, and B. Daneshfar, <span>“Detailed crop mapping using remote sensing data (crop data layers),”</span> J. Delince, Ed. Rome: Handbook of the Global Strategy to improve Agricultural; Rural Statistics (GSARS), 2017.</div>
</div>
<div id="ref-Defourny2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[5] </div>
<div class="csl-right-inline">P. Defourny, <span>“Land cover mapping and monitoring,”</span> J. Delincé, Ed. Rome: Handbook of the Global Strategy to improve Agricultural; Rural Statistics (GSARS), 2017.</div>
</div>
<div id="ref-Sarricolea2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[6] </div>
<div class="csl-right-inline">P. Sarricolea, M. Herrera-Ossandon, and Ó. Meseguer-Ruiz, <span>“Climatic regionalisation of continental Chile,”</span> <em>Journal of Maps</em>, vol. 13, no. 2, pp. 66–73, Nov. 2017, doi: <a href="https://doi.org/10.1080/17445647.2016.1259592">10.1080/17445647.2016.1259592</a>.</div>
</div>
<div id="ref-Santibanez2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[7] </div>
<div class="csl-right-inline">F. Santibáñez Q., P. V. Santibáñez, C. T. Caroca, and P. Gonzales, <em>Atlas agroclimático de Chile</em>. Santiago: Universidad de Chile, 2017.</div>
</div>
<div id="ref-Luebert2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[8] </div>
<div class="csl-right-inline">F. Luebert and P. Pliscoff, <em><a href="https://repositorio.uchile.cl/handle/2250/181641">Sinopsis bioclimática y vegetacional de Chile</a></em>, Segunda edición. Santiago de Chile: Editorial Universitaria, 2017.</div>
</div>
<div id="ref-Odepa2022b" class="csl-entry" role="listitem">
<div class="csl-left-margin">[9] </div>
<div class="csl-right-inline">ODEPA, <span>“Atlas rural de Chile,”</span> 2022. [Online]. Available: <a href="https://hdl.handle.net/20.500.12650/71509">https://hdl.handle.net/20.500.12650/71509</a>.</div>
</div>
<div id="ref-Beck2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">[10] </div>
<div class="csl-right-inline">H. E. Beck, N. E. Zimmermann, T. R. McVicar, N. Vergopolan, A. Berg, and E. F. Wood, <span>“Present and future Köppen-Geiger climate classification maps at 1-km resolution,”</span> <em>Scientific Data</em>, vol. 5, no. 1, p. 180214, Oct. 2018, doi: <a href="https://doi.org/10.1038/sdata.2018.214">10.1038/sdata.2018.214</a>.</div>
</div>
<div id="ref-Ojwang2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[11] </div>
<div class="csl-right-inline">G. O. Ojwang <em>et al.</em>, <span>“An integrated hierarchical classification and machine learning approach for mapping land use and land cover in complex social-ecological systems,”</span> <em>Frontiers in Remote Sensing</em>, vol. 4, p. 1188635, Mar. 2024, doi: <a href="https://doi.org/10.3389/frsen.2023.1188635">10.3389/frsen.2023.1188635</a>.</div>
</div>
<div id="ref-Chowdhury2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">[12] </div>
<div class="csl-right-inline">Md. S. Chowdhury, <span>“GIS based method for mapping actual LULC by combining seasonal LULCs,”</span> <em>MethodsX</em>, vol. 11, p. 102472, Dec. 2023, doi: <a href="https://doi.org/10.1016/j.mex.2023.102472">10.1016/j.mex.2023.102472</a>.</div>
</div>
<div id="ref-Myers2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[13] </div>
<div class="csl-right-inline">D. T. Myers, D. Jones, D. Oviedo-Vargas, J. P. Schmit, D. L. Ficklin, and X. Zhang, <span>“Seasonal variation in land cover estimates reveals sensitivities and opportunities for environmental models,”</span> <em>Hydrology and Earth System Sciences</em>, vol. 28, no. 23, p. 52955310, 2024, doi: <a href="https://doi.org/10.5194/hess-28-5295-2024">10.5194/hess-28-5295-2024</a>.</div>
</div>
<div id="ref-Guo2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">[14] </div>
<div class="csl-right-inline">Y. Guo, H. Xia, L. Pan, X. Zhao, and R. Li, <span>“Mapping the Northern Limit of Double Cropping Using a Phenology-Based Algorithm and Google Earth Engine,”</span> <em>Remote Sensing</em>, vol. 14, no. 4, p. 1004, Feb. 2022, doi: <a href="https://doi.org/10.3390/rs14041004">10.3390/rs14041004</a>.</div>
</div>
<div id="ref-Oetter2001" class="csl-entry" role="listitem">
<div class="csl-left-margin">[15] </div>
<div class="csl-right-inline">D. R. Oetter, W. B. Cohen, M. Berterretche, T. K. Maiersperger, and R. E. Kennedy, <span>“Land cover mapping in an agricultural setting using multiseasonal Thematic Mapper data,”</span> <em>Remote Sensing of Environment</em>, vol. 76, no. 2, pp. 139–155, May 2001, doi: <a href="https://doi.org/10.1016/S0034-4257(00)00202-9">10.1016/S0034-4257(00)00202-9</a>.</div>
</div>
<div id="ref-Wardlow2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">[16] </div>
<div class="csl-right-inline">B. D. Wardlow and S. L. Egbert, <span>“Large-area crop mapping using time-series MODIS 250 m NDVI data: An assessment for the U.S. Central Great Plains,”</span> <em>Remote Sensing of Environment</em>, vol. 112, no. 3, pp. 1096–1116, Mar. 2008, doi: <a href="https://doi.org/10.1016/j.rse.2007.07.019">10.1016/j.rse.2007.07.019</a>.</div>
</div>
<div id="ref-Zhu2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">[17] </div>
<div class="csl-right-inline">Z. Zhu and C. E. Woodcock, <span>“Continuous change detection and classification of land cover using all available <span>Landsat</span> data,”</span> <em>Remote Sensing of Environment</em>, vol. 144, pp. 152–171, 2014, doi: <a href="https://doi.org/10.1016/j.rse.2014.01.011">10.1016/j.rse.2014.01.011</a>.</div>
</div>
<div id="ref-Zhou2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[18] </div>
<div class="csl-right-inline">Y. Zhou <em>et al.</em>, <span>“Hierarchical classification for improving parcel-scale crop mapping using time-series Sentinel-1 data,”</span> <em>Journal of Environmental Management</em>, vol. 369, p. 122251, Oct. 2024, doi: <a href="https://doi.org/10.1016/j.jenvman.2024.122251">10.1016/j.jenvman.2024.122251</a>.</div>
</div>
<div id="ref-Zhou2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[19] </div>
<div class="csl-right-inline">Y. Zhou, J. Luo, L. Feng, Y. Yang, Y. Chen, and W. Wu, <span>“Long-short-term-memory-based crop classification using high-resolution optical images and multi-temporal SAR data,”</span> <em>GIScience &amp; Remote Sensing</em>, vol. 56, no. 8, pp. 1170–1191, Nov. 2019, doi: <a href="https://doi.org/10.1080/15481603.2019.1628412">10.1080/15481603.2019.1628412</a>.</div>
</div>
<div id="ref-Tahir2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[20] </div>
<div class="csl-right-inline">H. Md-Tahir <em>et al.</em>, <span>“Localized Crop Classification by NDVI Time Series Analysis of Remote Sensing Satellite Data; Applications for Mechanization Strategy and Integrated Resource Management,”</span> <em>AgriEngineering</em>, vol. 6, no. 3, pp. 2429–2444, Jul. 2024, doi: <a href="https://doi.org/10.3390/agriengineering6030142">10.3390/agriengineering6030142</a>.</div>
</div>
<div id="ref-Ullah2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">[21] </div>
<div class="csl-right-inline">S. Ullah <em>et al.</em>, <span>“Multi-Temporal and Multi-Resolution RGB UAV Surveys for Cost-Efficient Tree Species Mapping in an Afforestation Project,”</span> <em>Remote Sensing</em>, vol. 17, no. 6, p. 949, Mar. 2025, doi: <a href="https://doi.org/10.3390/rs17060949">10.3390/rs17060949</a>.</div>
</div>
<div id="ref-Schepaschenko2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[22] </div>
<div class="csl-right-inline">D. Schepaschenko <em>et al.</em>, <span>“Recent Advances in Forest Observation with Visual Interpretation of Very High-Resolution Imagery,”</span> <em>Surveys in Geophysics</em>, vol. 40, no. 4, pp. 839–862, Jul. 2019, doi: <a href="https://doi.org/10.1007/s10712-019-09533-z">10.1007/s10712-019-09533-z</a>.</div>
</div>
<div id="ref-Elmes2020a" class="csl-entry" role="listitem">
<div class="csl-left-margin">[23] </div>
<div class="csl-right-inline">A. Elmes <em>et al.</em>, <span>“Accounting for <span>Training Data Error</span> in <span>Machine Learning Applied</span> to <span>Earth Observations</span>,”</span> <em>Remote Sensing</em>, vol. 12, no. 6, p. 1034, 2020, doi: <a href="https://doi.org/10.3390/rs12061034">10.3390/rs12061034</a>.</div>
</div>
<div id="ref-Simoes2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">[24] </div>
<div class="csl-right-inline">R. Simoes <em>et al.</em>, <span>“Satellite image time series analysis for big earth observation data,”</span> <em>Remote Sensing</em>, vol. 13, no. 13, p. 2428, 2021, doi: <a href="https://doi.org/10.3390/rs13132428">10.3390/rs13132428</a>.</div>
</div>
<div id="ref-Camara2024b" class="csl-entry" role="listitem">
<div class="csl-left-margin">[25] </div>
<div class="csl-right-inline">G. Camara <em>et al.</em>, <span>“Bayesian <span>Inference</span> for <span>Post-Processing</span> of <span>Remote-Sensing Image Classification</span>,”</span> <em>Remote Sensing</em>, vol. 16, no. 23, p. 4572, 2024, doi: <a href="https://doi.org/10.3390/rs16234572">10.3390/rs16234572</a>.</div>
</div>
<div id="ref-Olofsson2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">[26] </div>
<div class="csl-right-inline">P. Olofsson, G. M. Foody, S. V. Stehman, and C. E. Woodcock, <span>“Making better use of accuracy data in land change studies: <span>Estimating</span> accuracy and area and quantifying uncertainty using stratified estimation,”</span> <em>Remote Sensing of Environment</em>, vol. 129, pp. 122–131, 2013, doi: <a href="https://doi.org/10.1016/j.rse.2012.10.031">10.1016/j.rse.2012.10.031</a>.</div>
</div>
<div id="ref-Olofsson2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">[27] </div>
<div class="csl-right-inline">P. Olofsson, G. M. Foody, M. Herold, S. V. Stehman, C. E. Woodcock, and M. A. Wulder, <span>“Good practices for estimating area and assessing accuracy of land change,”</span> <em>Remote Sensing of Environment</em>, vol. 148, pp. 42–57, 2014.</div>
</div>
<div id="ref-Santos2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">[28] </div>
<div class="csl-right-inline">L. A. Santos, K. Ferreira, M. Picoli, G. Camara, R. Zurita-Milla, and E.-W. Augustijn, <span>“Identifying <span>Spatiotemporal Patterns</span> in <span>Land Use</span> and <span>Cover Samples</span> from <span>Satellite Image Time Series</span>,”</span> <em>Remote Sensing</em>, vol. 13, no. 5, p. 974, 2021, doi: <a href="https://doi.org/10.3390/rs13050974">10.3390/rs13050974</a>.</div>
</div>
<div id="ref-Finegold2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">[29] </div>
<div class="csl-right-inline">Y. Finegold, E. Lindquist, R. D’Annuzio, and M. Sandker, <span>“Map <span>Accuracy Assessment</span> and <span>Area Estimation</span>: <span>A Practical Guide</span>,”</span> FAO, Rome, National forest monitoring assessment working paper No.46, 2016.</div>
</div>
<div id="ref-deSimone2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">[30] </div>
<div class="csl-right-inline">L. de Simone <em>et al.</em>, <span>“Quality control of training samples for agricultural statistics using <span>Earth</span> observation,”</span> <em>Statistical Journal of the IAOS</em>, p. 18747655251338033, 2025, doi: <a href="https://doi.org/10.1177/18747655251338033">10.1177/18747655251338033</a>.</div>
</div>
<div id="ref-Vesanto2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">[31] </div>
<div class="csl-right-inline">J. Vesanto and E. Alhoniemi, <span>“Clustering of the self-organizing map,”</span> <em>IEEE Transactions on Neural Networks</em>, vol. 11, no. 3, pp. 586–600, 2000, doi: <a href="https://doi.org/10.1109/72.846731">10.1109/72.846731</a>.</div>
</div>
<div id="ref-Johnson2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[32] </div>
<div class="csl-right-inline">J. M. Johnson and T. M. Khoshgoftaar, <span>“Survey on deep learning with class imbalance,”</span> <em>Journal of Big Data</em>, vol. 6, no. 1, p. 27, 2019, doi: <a href="https://doi.org/10.1186/s40537-019-0192-5">10.1186/s40537-019-0192-5</a>.</div>
</div>
<div id="ref-Foody2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">[33] </div>
<div class="csl-right-inline">G. M. Foody, <span>“Status of land cover classification accuracy assessment,”</span> <em>Remote Sensing of Environment</em>, vol. 80, no. 1, pp. 185–201, Apr. 2002, doi: <a href="https://doi.org/10.1016/s0034-4257(01)00295-4">10.1016/s0034-4257(01)00295-4</a>.</div>
</div>
<div id="ref-Radoux2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">[34] </div>
<div class="csl-right-inline">J. Radoux, C. Lamarche, E. Van Bogaert, S. Bontemps, C. Brockmann, and P. Defourny, <span>“Automated Training Sample Extraction for Global Land Cover Mapping,”</span> <em>Remote Sensing</em>, vol. 6, no. 5, pp. 3965–3987, May 2014, doi: <a href="https://doi.org/10.3390/rs6053965">10.3390/rs6053965</a>.</div>
</div>
<div id="ref-Zhao2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">[35] </div>
<div class="csl-right-inline">Y. Zhao <em>et al.</em>, <span>“Detailed dynamic land cover mapping of <span>Chile</span>: <span>Accuracy</span> improvement by integrating multi-temporal data,”</span> <em>Remote Sensing of Environment</em>, vol. 183, pp. 170–185, 2016, doi: <a href="https://doi.org/10.1016/j.rse.2016.05.016">10.1016/j.rse.2016.05.016</a>.</div>
</div>
<div id="ref-Galleguillos2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[36] </div>
<div class="csl-right-inline">M. Galleguillos, A. Ceballos-Comisso, F. Gimeno, and M. Zambrano-Bigiarini, <span>“<span>CLDynamicLandCover</span>.”</span> Zenodo, 2024, doi: <a href="https://doi.org/10.5281/zenodo.13153631">10.5281/zenodo.13153631</a>.</div>
</div>
<div id="ref-FAO2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[37] </div>
<div class="csl-right-inline">FAO, <span>“Handbook on remote sensing for agricultural statistics,”</span> 2017, [Online]. Available: <a href="https://openknowledge.fao.org/handle/20.500.14283/ca6394en">https://openknowledge.fao.org/handle/20.500.14283/ca6394en</a>.</div>
</div>
<div id="ref-tasktea" class="csl-entry" role="listitem">
<div class="csl-left-margin">[38] </div>
<div class="csl-right-inline">
<span>“Task team on earth observation data <span></span> UN-CEBD.”</span> [Online]. Available: <a href="https://unstats.un.org/bigdata/task-teams/earth-observation/index.cshtml">https://unstats.un.org/bigdata/task-teams/earth-observation/index.cshtml</a>.</div>
</div>
<div id="ref-Zhang2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[39] </div>
<div class="csl-right-inline">H. Zhang <em>et al.</em>, <span>“Mapping annual 10-m soybean cropland with spatiotemporal sample migration,”</span> <em>Scientific Data</em>, vol. 11, no. 1, May 2024, doi: <a href="https://doi.org/10.1038/s41597-024-03273-5">10.1038/s41597-024-03273-5</a>.</div>
</div>
<div id="ref-Ren2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">[40] </div>
<div class="csl-right-inline">T. Ren, H. Xu, X. Cai, S. Yu, and J. Qi, <span>“Smallholder Crop Type Mapping and Rotation Monitoring in Mountainous Areas with Sentinel-1/2 Imagery,”</span> <em>Remote Sensing</em>, vol. 14, no. 3, p. 566, Jan. 2022, doi: <a href="https://doi.org/10.3390/rs14030566">10.3390/rs14030566</a>.</div>
</div>
<div id="ref-Wasniewski2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">[41] </div>
<div class="csl-right-inline">A. Waśniewski, A. Hościło, and M. Chmielewska, <span>“Can a Hierarchical Classification of Sentinel-2 Data Improve Land Cover Mapping?”</span> <em>Remote Sensing</em>, vol. 14, no. 4, p. 989, Feb. 2022, doi: <a href="https://doi.org/10.3390/rs14040989">10.3390/rs14040989</a>.</div>
</div>
<div id="ref-Wardlow2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">[42] </div>
<div class="csl-right-inline">B. D. Wardlow, S. L. Egbert, and J. H. Kastens, <span>“Analysis of time-series MODIS 250m vegetation index data for crop classification in the U.S. Central Great Plains,”</span> <em>Remote Sensing of Environment</em>, vol. 108, no. 3, pp. 290–310, Jun. 2007, doi: <a href="https://doi.org/10.1016/j.rse.2006.11.021">10.1016/j.rse.2006.11.021</a>.</div>
</div>
<div id="ref-Zhang2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">[43] </div>
<div class="csl-right-inline">H. Zhang, J. He, S. Chen, Y. Zhan, Y. Bai, and Y. Qin, <span>“Comparing Three Methods of Selecting Training Samples in Supervised Classification of Multispectral Remote Sensing Images,”</span> <em>Sensors</em>, vol. 23, no. 20, p. 8530, Oct. 2023, doi: <a href="https://doi.org/10.3390/s23208530">10.3390/s23208530</a>.</div>
</div>
<div id="ref-Kavhu2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">[44] </div>
<div class="csl-right-inline">B. Kavhu, Z. E. Mashimbye, and L. Luvuno, <span>“Climate-Based Regionalization and Inclusion of Spectral Indices for Enhancing Transboundary Land-Use/Cover Classification Using Deep Learning and Machine Learning,”</span> <em>Remote Sensing</em>, vol. 13, no. 24, p. 5054, Dec. 2021, doi: <a href="https://doi.org/10.3390/rs13245054">10.3390/rs13245054</a>.</div>
</div>
<div id="ref-Dong2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">[45] </div>
<div class="csl-right-inline">S. Dong, H. Guo, Z. Chen, Y. Pan, and B. Gao, <span>“Spatial Stratification Method for the Sampling Design of LULC Classification Accuracy Assessment: A Case Study in Beijing, China,”</span> <em>Remote Sensing</em>, vol. 14, no. 4, p. 865, Feb. 2022, doi: <a href="https://doi.org/10.3390/rs14040865">10.3390/rs14040865</a>.</div>
</div>
<div id="ref-Loveland2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">[46] </div>
<div class="csl-right-inline">T. R. Loveland and J. M. Merchant, <span>“Ecoregions and Ecoregionalization: Geographical and Ecological Perspectives,”</span> <em>Environmental Management</em>, vol. 34, no. S1, pp. S1–S13, Apr. 2004, doi: <a href="https://doi.org/10.1007/s00267-003-5181-x">10.1007/s00267-003-5181-x</a>.</div>
</div>
<div id="ref-Hansen2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">[47] </div>
<div class="csl-right-inline">M. C. Hansen <em>et al.</em>, <span>“High-<span>Resolution Global Maps</span> of 21st-<span>Century Forest Cover Change</span>,”</span> <em>Science</em>, vol. 342, no. 6160, pp. 850–853, 2013.</div>
</div>
<div id="ref-Luebert2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">[48] </div>
<div class="csl-right-inline">F. Luebert and P. Pliscoff, <span>“Clasificación de pisos de vegetación y análisis de representatividad ecológica de áreas propuestas para la protección en la ecorregión valdiviana,”</span> Valdivia, Chile, Dec. 2004. [Online]. Available: <a href="https://wwflac.awsassets.panda.org/downloads/informe_ecorregion_valdiviana_luebert_pliscoff.pdf#:~:text=pero%20se%20manifiestan%20en%20la,azonales%20de%20diferentes%20tipos%2C%20especialmente">https://wwflac.awsassets.panda.org/downloads/informe_ecorregion_valdiviana_luebert_pliscoff.pdf#:~:text=pero%20se%20manifiestan%20en%20la,azonales%20de%20diferentes%20tipos%2C%20especialmente</a>.</div>
</div>
<div id="ref-Bilucan2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">[49] </div>
<div class="csl-right-inline">F. Bilücan and T. Kavzoglu, <span>“The effect of auxiliary data (slope, aspect and elevation ) on land use/land cover classification accuracy of sentinel-2A image using random forest classifier,”</span> May 2021.</div>
</div>
<div id="ref-PachecoPrado2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[50] </div>
<div class="csl-right-inline">D. Pacheco-Prado, E. Bravo-López, and L. Á. Ruiz, <span>“Mapping Polylepis Forest Using Sentinel, PlanetScope Images, and Topographical Features with Machine Learning,”</span> <em>Remote Sensing</em>, vol. 16, no. 22, p. 4271, Nov. 2024, doi: <a href="https://doi.org/10.3390/rs16224271">10.3390/rs16224271</a>.</div>
</div>
<div id="ref-Pizarro2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">[51] </div>
<div class="csl-right-inline">S. E. Pizarro, N. G. Pricope, D. Vargas-Machuca, O. Huanca, and J. Ñaupari, <span>“Mapping Land Cover Types for Highland Andean Ecosystems in Peru Using Google Earth Engine,”</span> <em>Remote Sensing</em>, vol. 14, no. 7, p. 1562, Mar. 2022, doi: <a href="https://doi.org/10.3390/rs14071562">10.3390/rs14071562</a>.</div>
</div>
<div id="ref-Mujahid2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[52] </div>
<div class="csl-right-inline">M. Mujahid <em>et al.</em>, <span>“Data oversampling and imbalanced datasets: an investigation of performance for machine learning and feature engineering,”</span> <em>Journal of Big Data</em>, vol. 11, no. 1, Jun. 2024, doi: <a href="https://doi.org/10.1186/s40537-024-00943-4">10.1186/s40537-024-00943-4</a>.</div>
</div>
<div id="ref-May2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">[53] </div>
<div class="csl-right-inline">R. J. May, H. R. Maier, and G. C. Dandy, <span>“Data splitting for artificial neural networks using SOM-based stratified sampling,”</span> <em>Neural Networks</em>, vol. 23, no. 2, pp. 283–294, Mar. 2010, doi: <a href="https://doi.org/10.1016/j.neunet.2009.11.009">10.1016/j.neunet.2009.11.009</a>.</div>
</div>
<div id="ref-RuizMoreno2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">[54] </div>
<div class="csl-right-inline">S. Ruiz-Moreno, A. Núñez-Reyes, A. García-Cantalapiedra, and F. Pavón, <span>“Prototype generation method using a growing self-organizing map applied to the banking sector,”</span> <em>Neural Computing and Applications</em>, vol. 35, no. 24, pp. 17579–17597, Aug. 2023, doi: <a href="https://doi.org/10.1007/s00521-023-08630-w">10.1007/s00521-023-08630-w</a>.</div>
</div>
<div id="ref-Douzas2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[55] </div>
<div class="csl-right-inline">G. Douzas, F. Bacao, J. Fonseca, and M. Khudinyan, <span>“Imbalanced Learning in Land Cover Classification: Improving Minority Classes<span>’</span> Prediction Accuracy Using the Geometric SMOTE Algorithm,”</span> <em>Remote Sensing</em>, vol. 11, no. 24, p. 3040, Dec. 2019, doi: <a href="https://doi.org/10.3390/rs11243040">10.3390/rs11243040</a>.</div>
</div>
<div id="ref-Xu2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[56] </div>
<div class="csl-right-inline">W. Xu <em>et al.</em>, <span>“Mapping Fruit-Tree Plantation Using Sentinel-1/2 Time Series Images with Multi-Index Entropy Weighting Dynamic Time Warping Method,”</span> <em>Remote Sensing</em>, vol. 16, no. 18, p. 3390, Sep. 2024, doi: <a href="https://doi.org/10.3390/rs16183390">10.3390/rs16183390</a>.</div>
</div>
<div id="ref-Douzas2019a" class="csl-entry" role="listitem">
<div class="csl-left-margin">[57] </div>
<div class="csl-right-inline">G. Douzas and F. Bacao, <span>“Geometric SMOTE a geometrically enhanced drop-in replacement for SMOTE,”</span> <em>Information Sciences</em>, vol. 501, pp. 118–135, Oct. 2019, doi: <a href="https://doi.org/10.1016/j.ins.2019.06.007">10.1016/j.ins.2019.06.007</a>.</div>
</div>
<div id="ref-EuropeanCommission2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">[58] </div>
<div class="csl-right-inline">European Commission, <em><a href="https://doi.org/10.2785/616374">European Statistical System Handbook for Quality and Metadata Reports</a></em>, 2021 re-edition. Luxembourg: Publications Office of the European Union, 2021.</div>
</div>
<div id="ref-Russwurm2020a" class="csl-entry" role="listitem">
<div class="csl-left-margin">[59] </div>
<div class="csl-right-inline">M. Rußwurm and M. Körner, <span>“Self-attention for raw optical <span>Satellite Time Series Classification</span>,”</span> <em>ISPRS Journal of Photogrammetry and Remote Sensing</em>, vol. 169, pp. 421–435, 2020, doi: <a href="https://doi.org/10.1016/j.isprsjprs.2020.06.006">10.1016/j.isprsjprs.2020.06.006</a>.</div>
</div>
<div id="ref-Tsuchiya2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">[60] </div>
<div class="csl-right-inline">Y. Tsuchiya and R. Sonobe, <span>“Crop Classification Using Time-Series Sentinel-1 SAR Data: A Comparison of LSTM, GRU, and TCN with Attention,”</span> <em>Remote Sensing</em>, vol. 17, no. 12, p. 2095, Jun. 2025, doi: <a href="https://doi.org/10.3390/rs17122095">10.3390/rs17122095</a>.</div>
</div>
<div id="ref-Wang2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">[61] </div>
<div class="csl-right-inline">H. Wang <em>et al.</em>, <span>“Cropformer: A new generalized deep learning classification approach for multi-scenario crop classification,”</span> <em>Frontiers in Plant Science</em>, vol. 14, Mar. 2023, doi: <a href="https://doi.org/10.3389/fpls.2023.1130659">10.3389/fpls.2023.1130659</a>.</div>
</div>
<div id="ref-Zhong2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[62] </div>
<div class="csl-right-inline">L. Zhong, L. Hu, and H. Zhou, <span>“Deep learning based multi-temporal crop classification,”</span> <em>Remote Sensing of Environment</em>, vol. 221, pp. 430–443, Feb. 2019, doi: <a href="https://doi.org/10.1016/j.rse.2018.11.032">10.1016/j.rse.2018.11.032</a>.</div>
</div>
<div id="ref-Pelletier2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[63] </div>
<div class="csl-right-inline">C. Pelletier, G. I. Webb, and F. Petitjean, <span>“Temporal <span>Convolutional Neural Network</span> for the <span>Classification</span> of <span>Satellite Image Time Series</span>,”</span> <em>Remote Sensing</em>, vol. 11, no. 5, 2019.</div>
</div>
<div id="ref-Mcnairn2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">[64] </div>
<div class="csl-right-inline">H. McNairn, C. Champagne, J. Shang, D. Holmstrom, and G. Reichert, <span>“Integration of optical and Synthetic Aperture Radar (SAR) imagery for delivering operational annual crop inventories,”</span> <em>ISPRS Journal of Photogrammetry and Remote Sensing</em>, vol. 64, no. 5, pp. 434–449, Sep. 2009, doi: <a href="https://doi.org/10.1016/j.isprsjprs.2008.07.006">10.1016/j.isprsjprs.2008.07.006</a>.</div>
</div>
<div id="ref-PenaBarragan2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">[65] </div>
<div class="csl-right-inline">J. M. Peña-Barragán, M. K. Ngugi, R. E. Plant, and J. Six, <span>“Object-based crop identification using multiple vegetation indices, textural features and crop phenology,”</span> <em>Remote Sensing of Environment</em>, vol. 115, no. 6, pp. 1301–1316, Jun. 2011, doi: <a href="https://doi.org/10.1016/j.rse.2011.01.009">10.1016/j.rse.2011.01.009</a>.</div>
</div>
<div id="ref-Ambrosio2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[66] </div>
<div class="csl-right-inline">L. Ambrosio <em>et al.</em>, <span>“<a href="https://doi.org/10.5772/intechopen.1004409">Earth observation applications to agricultural and environmental statistics</a>,”</span> IntechOpen, 2024.</div>
</div>
<div id="ref-Rojas2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[67] </div>
<div class="csl-right-inline">O. Rojas, Natural Resources Officer, and FAO SLM/CBC, <span>“Early warning systems and crop yield estimation,”</span> J. Delincé, Ed. Rome: Handbook of the Global Strategy to improve Agricultural; Rural Statistics (GSARS), 2017.</div>
</div>
<div id="ref-Achard2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[68] </div>
<div class="csl-right-inline">F. Achard, Y. M. Malheiros de Oliveira, and D. Mollicone, <span>“Monitoring forest cover and deforestation,”</span> J. Delincé, Ed. Rome: Handbook of the Global Strategy to improve Agricultural; Rural Statistics (GSARS), 2017.</div>
</div>
<div id="ref-Grippa2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[69] </div>
<div class="csl-right-inline">T. Grippa <em>et al.</em>, <span>“Improving Urban Population Distribution Models with Very-High Resolution Satellite Information,”</span> <em>Data</em>, vol. 4, no. 1, p. 13, Jan. 2019, doi: <a href="https://doi.org/10.3390/data4010013">10.3390/data4010013</a>.</div>
</div>
<div id="ref-Ozesmi2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">[70] </div>
<div class="csl-right-inline">S. L. Ozesmi and M. E. Bauer, <span>“Satellite remote sensing of wetlands,”</span> <em>Wetlands Ecology and Management</em>, vol. 10, no. 5, pp. 381–402, 2002, doi: <a href="https://doi.org/10.1023/a:1020908432489">10.1023/a:1020908432489</a>.</div>
</div>
<div id="ref-Liu2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[71] </div>
<div class="csl-right-inline">L. Liu, J. Chen, P. Fieguth, G. Zhao, R. Chellappa, and M. Pietikäinen, <span>“From BoW to CNN: Two Decades of Texture Representation for Texture Classification,”</span> <em>International Journal of Computer Vision</em>, vol. 127, no. 1, pp. 74–109, Jan. 2019, doi: <a href="https://doi.org/10.1007/s11263-018-1125-z">10.1007/s11263-018-1125-z</a>.</div>
</div>
</div>
</section><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><hr>
<ol>
<li id="fn1"><p>National Forestry Corporation (CONAF by its acronym in Spanish).<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p>For the entire MZ4, 16,190 samples (26% of the dataset) were flagged as potentially noisy. Removing them would increase class imbalance between 0.3% and 9%, and would even result in the complete elimination of samples from the <code>Bare_rock</code> class.<a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3"><p>When considering the entire MZ4, the three most common labels (<code>Mature_plantation</code>, <code>Perennial_fores</code>t, and <code>Deciduous_fruit_trees</code>) account for 24% of the total sample units. Meanwhile, the three least frequent classes (<code>Bare_rocks</code>, <code>Seasonal_snow_area</code>, and <code>Wetlands</code>) make up just 1.8% of the dataset.<a href="#fnref3" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol></section></main><!-- /main --><script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><nav class="page-navigation"><div class="nav-page nav-page-previous">
      <a href="./ct_zimbabwe.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Crop classification in Zimbabwe</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./ct_digital_earth_africa.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Agricultural Mapping with Digital Earth Africa</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer"><div class="nav-footer">
    <div class="nav-footer-left">Edited by Gilberto Camara, Lorenzo de Simone and Ronald Jansen.</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">This book was built with <a href="https://quarto.org/">Quarto</a>.</div>
  </div>
</footer>


</body></html>